<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Stephen Pederson &amp; Caitlin Abbott" />


<title>Tr1: Enrichment Analysis</title>

<script src="site_libs/header-attrs-2.21/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/core-js-2.5.3/shim.min.js"></script>
<script src="site_libs/react-17.0.0/react.min.js"></script>
<script src="site_libs/react-17.0.0/react-dom.min.js"></script>
<script src="site_libs/reactwidget-1.0.0/react-tools.js"></script>
<script src="site_libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<link href="site_libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="site_libs/reactable-binding-0.4.4/reactable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tr1-RNA-Sequencing</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="dge.html">DGE Analysis</a>
</li>
<li>
  <a href="enrichment.html">Enrichment Analysis</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Tr1: Enrichment Analysis</h1>
<h4 class="author">Stephen Pederson &amp; Caitlin Abbott</h4>
<h4 class="date">01 May, 2023</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-05-01
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>Tr1-RNA-Sequencing-/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20221025code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20221025)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20221025code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20221025)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomsmpedTr1RNASequencingtreeef7ce0cd2027036772c132031eae19539f2b0bddtargetblankef7ce0ca">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/tree/ef7ce0cd2027036772c132031eae19539f2b0bdd" target="_blank">ef7ce0c</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomsmpedTr1RNASequencingtreeef7ce0cd2027036772c132031eae19539f2b0bddtargetblankef7ce0ca"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/tree/ef7ce0cd2027036772c132031eae19539f2b0bdd" target="_blank">ef7ce0c</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Unstaged changes:
    Modified:   analysis/enrichment.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/enrichment.Rmd</code>) and HTML
(<code>docs/enrichment.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/3933b927fbea65a90c92a51bcbee49f26fa1b890/analysis/enrichment.Rmd" target="_blank">3933b92</a>
</td>
<td>
steveped
</td>
<td>
2022-11-18
</td>
<td>
Corrected section ids
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/smped/Tr1-RNA-Sequencing-/3933b927fbea65a90c92a51bcbee49f26fa1b890/docs/enrichment.html" target="_blank">3933b92</a>
</td>
<td>
steveped
</td>
<td>
2022-11-18
</td>
<td>
Corrected section ids
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/smped/Tr1-RNA-Sequencing-/3a59bc588052927680289ec0a029cb568d1f8d03/docs/enrichment.html" target="_blank">3a59bc5</a>
</td>
<td>
steveped
</td>
<td>
2022-11-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/36a1cb886f7af393e21a64260818552ac4cc0d36/analysis/enrichment.Rmd" target="_blank">36a1cb8</a>
</td>
<td>
steveped
</td>
<td>
2022-11-18
</td>
<td>
Added enrichment tables
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/smped/Tr1-RNA-Sequencing-/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/enrichment.html" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/68c392fc4a27599c299ff65e989e6136c26b84aa/analysis/enrichment.Rmd" target="_blank">68c392f</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
<td>
Added down &amp; up genes separately
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/smped/Tr1-RNA-Sequencing-/f2ee30ed28f0c3007704577393163e951327740c/docs/enrichment.html" target="_blank">f2ee30e</a>
</td>
<td>
steveped
</td>
<td>
2022-11-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/61ec31fdbfcf7d35b508a533f329f98933a65299/analysis/enrichment.Rmd" target="_blank">61ec31f</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-11-01
</td>
<td>
Added enrichment testing
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/smped/Tr1-RNA-Sequencing-/61ec31fdbfcf7d35b508a533f329f98933a65299/docs/enrichment.html" target="_blank">61ec31f</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-11-01
</td>
<td>
Added enrichment testing
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0892674261bb82edea954d3b372ee1a157b30381/analysis/enrichment.Rmd" target="_blank">0892674</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-10-26
</td>
<td>
Started adding heatmaps
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/8745ba2c539e132ad5328b7b767920569b16577c/analysis/enrichment.Rmd" target="_blank">8745ba2</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-10-26
</td>
<td>
Shifted to workflowr
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>knitr::opts_chunk$set(
  warning = FALSE, message = FALSE,
  fig.width = 10, fig.height = 8,
  dev = c(&quot;png&quot;, &quot;pdf&quot;)
)</code></pre>
<pre class="r"><code>library(tidyverse)
library(limma)
library(edgeR)
library(RColorBrewer)
library(pheatmap)
library(magrittr)
library(clusterProfiler)
library(org.Mm.eg.db)
library(reactable)
library(htmltools)
library(glue)
library(scales)
with_tooltip &lt;- function(value, width = 30) {
  tags$span(title = value, str_trunc(value, width))
}</code></pre>
<pre class="r"><code>top_tables &lt;- read_rds(here::here(&quot;output&quot;, &quot;top_tables.rds&quot;))
dgeFilt &lt;- read_rds(here::here(&quot;output&quot;, &quot;dgeFilt.rds&quot;))</code></pre>
<pre class="r"><code>universe &lt;- dgeFilt$genes$entrezid %&gt;%
  unlist() %&gt;%
  extract(!is.na(.)) %&gt;% #keep non NA gene
  unique() %&gt;%
  as.character()</code></pre>
<div id="enrichment-results" class="section level2 tabset">
<h2 class="tabset">Enrichment Results</h2>
<pre class="r"><code>ego_results &lt;- top_tables %&gt;% 
  lapply(
    function(x) {
      de &lt;- dplyr::filter(x, adj.P.Val &lt; 0.05, abs(logFC) &gt; 1)
      ids &lt;- unique(unlist(de$entrezid))
      enrichGO(
        ids,
        universe = universe,
        OrgDb = org.Mm.eg.db,
        ont = &quot;ALL&quot;,
        keyType = &quot;ENTREZID&quot;,
        pAdjustMethod = &quot;bonferroni&quot;,
        pvalueCutoff = 0.05,
        qvalueCutoff = 0.05,
        readable = TRUE
      )
    }
  )</code></pre>
<p>Enrichment testing was performed using all three available Gene
Ontologies and the function <code>enrichGO()</code> from the package
<code>clusterProfiler</code>. Ontologies were only considered to be
enriched amongst the differentially expressed genes if a
Bonferroni-adjusted p-value &lt; 0.05 was returned during enrichment
testing, with no regard to the sign of fold change.</p>
<div id="dn-vs.-lag3" class="section level3">
<h3>DN Vs. Lag3<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$DNvLAG3) +
  ggtitle(expression(paste(&quot;DN Vs. Lag&quot;, 3^{textstyle(&quot;+&quot;)}))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-dn-lag3-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-dn-lag3-1">
Past versions of dotplot-dn-lag3-1.png
</button>
</p>
<div id="fig-dotplot-dn-lag3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/f2ee30ed28f0c3007704577393163e951327740c/docs/figure/enrichment.Rmd/dotplot-dn-lag3-1.png" target="_blank">f2ee30e</a>
</td>
<td>
steveped
</td>
<td>
2022-11-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/61ec31fdbfcf7d35b508a533f329f98933a65299/docs/figure/enrichment.Rmd/dotplot-dn-lag3-1.png" target="_blank">61ec31f</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-11-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dn-vs.-dp" class="section level3">
<h3>DN Vs. DP</h3>
<pre class="r"><code>dotplot(ego_results$DNvDP) +
  ggtitle(&quot;DN Vs. DP&quot;) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-dn-dp-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-dn-dp-1">
Past versions of dotplot-dn-dp-1.png
</button>
</p>
<div id="fig-dotplot-dn-dp-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/f2ee30ed28f0c3007704577393163e951327740c/docs/figure/enrichment.Rmd/dotplot-dn-dp-1.png" target="_blank">f2ee30e</a>
</td>
<td>
steveped
</td>
<td>
2022-11-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/61ec31fdbfcf7d35b508a533f329f98933a65299/docs/figure/enrichment.Rmd/dotplot-dn-dp-1.png" target="_blank">61ec31f</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-11-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dn-vs.-cd49b" class="section level3">
<h3>DN Vs. Cd49b<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$DNvCD49b) +
  ggtitle(expression(paste(&quot;DN Vs. Cd49&quot;, b^{textstyle(&quot;+&quot;)}))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-dn-cd49-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-dn-cd49-1">
Past versions of dotplot-dn-cd49-1.png
</button>
</p>
<div id="fig-dotplot-dn-cd49-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/f2ee30ed28f0c3007704577393163e951327740c/docs/figure/enrichment.Rmd/dotplot-dn-cd49-1.png" target="_blank">f2ee30e</a>
</td>
<td>
steveped
</td>
<td>
2022-11-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/61ec31fdbfcf7d35b508a533f329f98933a65299/docs/figure/enrichment.Rmd/dotplot-dn-cd49-1.png" target="_blank">61ec31f</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-11-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="lag3-vs-cd49" class="section level3">
<h3>Lag3<sup>+</sup> Vs Cd49<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$LAG3vCD49b) +
  ggtitle(
    expression(
      paste(&quot;Lag&quot;, 3^{textstyle(&quot;+&quot;)}, &quot; Vs. CD49&quot;, b^{textstyle(&quot;+&quot;)})
    )
  ) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-lag3-cd49-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-lag3-cd49-1">
Past versions of dotplot-lag3-cd49-1.png
</button>
</p>
<div id="fig-dotplot-lag3-cd49-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/f2ee30ed28f0c3007704577393163e951327740c/docs/figure/enrichment.Rmd/dotplot-lag3-cd49-1.png" target="_blank">f2ee30e</a>
</td>
<td>
steveped
</td>
<td>
2022-11-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/61ec31fdbfcf7d35b508a533f329f98933a65299/docs/figure/enrichment.Rmd/dotplot-lag3-cd49-1.png" target="_blank">61ec31f</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-11-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="lag3-vs.-dp" class="section level3">
<h3>Lag3<sup>+</sup> Vs. DP</h3>
<pre class="r"><code>dotplot(ego_results$LAG3vDP) +
  ggtitle(expression(paste(&quot;Lag&quot;, 3^{textstyle(&quot;+&quot;)}, &quot; Vs. DP&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-lag3-dp-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-lag3-dp-1">
Past versions of dotplot-lag3-dp-1.png
</button>
</p>
<div id="fig-dotplot-lag3-dp-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/f2ee30ed28f0c3007704577393163e951327740c/docs/figure/enrichment.Rmd/dotplot-lag3-dp-1.png" target="_blank">f2ee30e</a>
</td>
<td>
steveped
</td>
<td>
2022-11-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/61ec31fdbfcf7d35b508a533f329f98933a65299/docs/figure/enrichment.Rmd/dotplot-lag3-dp-1.png" target="_blank">61ec31f</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-11-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cd49b-vs.-dp" class="section level3">
<h3>Cd49b<sup>+</sup> Vs. DP</h3>
<pre class="r"><code>dotplot(ego_results$CD49bvDP) +
  ggtitle(expression(paste(&quot;Cd49&quot;, b^{textstyle(&quot;+&quot;)}, &quot; Vs. DP&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-cd49-dp-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-cd49-dp-1">
Past versions of dotplot-cd49-dp-1.png
</button>
</p>
<div id="fig-dotplot-cd49-dp-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/f2ee30ed28f0c3007704577393163e951327740c/docs/figure/enrichment.Rmd/dotplot-cd49-dp-1.png" target="_blank">f2ee30e</a>
</td>
<td>
steveped
</td>
<td>
2022-11-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/61ec31fdbfcf7d35b508a533f329f98933a65299/docs/figure/enrichment.Rmd/dotplot-cd49-dp-1.png" target="_blank">61ec31f</a>
</td>
<td>
Steve Pederson
</td>
<td>
2022-11-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="enrichment-tables" class="section level2 tabset">
<h2 class="tabset">Enrichment Tables</h2>
<pre class="r"><code>htmltools::tagList(
  names(ego_results) %&gt;% 
    lapply(
      function(x) {
        df &lt;- ego_results[[x]] %&gt;% 
          as_tibble() %&gt;% 
          dplyr::filter(p.adjust &lt; 0.05) %&gt;% 
          mutate(
            geneID = str_replace_all(geneID, &quot;\\/&quot;, &quot;; &quot;),
            `% DE` = vapply(GeneRatio, function(x) eval(parse(text = x)), numeric(1)),
            `% BG` = vapply(BgRatio, function(x) eval(parse(text = x)), numeric(1))
          ) %&gt;% 
          dplyr::select(ONTOLOGY, ID, Description, Count, starts_with(&quot;%&quot;), p.adjust, geneID) %&gt;% 
          arrange(p.adjust) %&gt;% 
          distinct(ONTOLOGY, geneID, .keep_all = TRUE) 
        
        if (nrow(df)) {
          cp &lt;- glue(
            &quot;All {nrow(df)} ontologies considered enriched in the set of &quot;,
            length(unique(unlist(dplyr::filter(top_tables[[x]], DE)$entrezid))),
            &quot; DE genes mapped to EntrezIDs from the comparison {x}. Enrichment &quot;,
            &quot;testing was performed using Fisher&#39;s Exact Test compared to the set of &quot;,
            comma(length(unique(universe))), 
            &quot; unique EntrezIDs mapped to genes considered as detectable. &quot;,
            &quot;The percentage of nonDE genes mapped to the ontology are also given. &quot;,
            &quot;P-values are Bonferroni adjusted and all pathways are considered enriched &quot;,
            &quot;using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your &quot;, 
            &quot;mouse over the final column. &quot;,
            &quot;Where the identical genes map to multiple terms within each ontology, only &quot;, 
            &quot;the term with the lowest p-value (i.e. strongest enrichment) is shown. &quot;,
            &quot;As a result, some of the pathways seen in the above dotplots may not be &quot;,
            &quot;presented in these tables.&quot;
          )
          htmltools::div(
            htmltools::div(
              id = x, 
              class=&quot;section level3&quot;,
              htmltools::h3(class = &quot;tabset&quot;, x),
              htmltools::tags$em(cp),
              df %&gt;% 
                reactable(
                  filterable = TRUE,
                  columns = list(
                    ONTOLOGY = colDef(name = &quot;Ontology&quot;, maxWidth = 100),
                    ID = colDef(name = &quot;GO ID&quot;, maxWidth = 125),
                    Description = colDef(maxWidth = 250),
                    Count = colDef(name = &quot;N&lt;sub&gt;DE&lt;/sub&gt;&quot;, html = TRUE, maxWidth = 80),
                    `% DE` = colDef(format = colFormat(percent = TRUE, digits = 1), maxWidth = 85),
                    `% BG` = colDef(format = colFormat(percent = TRUE, digits = 1), maxWidth = 85),
                    p.adjust = colDef(
                      name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE, maxWidth = 100,
                      cell = function(value) {
                        fm &lt;- ifelse(value &lt; 0.001, &quot;%.2e&quot;, &quot;%.3f&quot;)
                        sprintf(fm, value)
                      }
                    ),
                    geneID = colDef(
                      name = &quot;DE Genes&quot;,
                      cell = function(value) with_tooltip(value, width = 100)
                    )
                  )
                )
            )
          )
        } else {
          cat(glue(&quot;No enrichment was found for {x}\n\n&quot;))
        }
      }
    )
)</code></pre>
<div>
<div id="DNvLAG3" class="section level3">
<h3 class="tabset">DNvLAG3</h3>
<em>All 55 ontologies considered enriched in the set of 73 DE genes mapped to EntrezIDs from the comparison DNvLAG3. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-9792121a243fec82197d" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-9792121a243fec82197d">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["CC","BP","CC","CC","MF","BP","CC","MF","BP","BP","CC","MF","CC","BP","MF","BP","BP","BP","BP","BP","BP","BP","BP","BP","BP","MF","BP","CC","CC","BP","BP","BP","BP","BP","BP","CC","BP","BP","BP","BP","MF","BP","BP","BP","MF","BP","BP","CC","BP","MF","BP","BP","BP","BP","BP"],"ID":["GO:0005615","GO:0006954","GO:0009986","GO:0009897","GO:0005125","GO:0034341","GO:0098797","GO:0008009","GO:0009617","GO:0022407","GO:0098552","GO:0023026","GO:0043235","GO:0002250","GO:0005126","GO:0032496","GO:0050778","GO:0071347","GO:0033993","GO:0070098","GO:0002694","GO:0098609","GO:0051249","GO:0050863","GO:0071222","GO:0038023","GO:0002768","GO:0042611","GO:0098802","GO:0002253","GO:0022409","GO:0030595","GO:0002757","GO:0030593","GO:0002449","GO:0099634","GO:0002443","GO:0019886","GO:0050870","GO:0002429","GO:0030296","GO:0042110","GO:0032103","GO:0051251","GO:0004888","GO:0071396","GO:0034110","GO:0005771","GO:1903131","GO:0003823","GO:0032722","GO:1903037","GO:0070372","GO:0043408","GO:0031663"],"Description":["extracellular space","inflammatory response","cell surface","external side of plasma membrane","cytokine activity","response to type II interferon","plasma membrane protein complex","chemokine activity","response to bacterium","regulation of cell-cell adhesion","side of membrane","MHC class II protein complex binding","receptor complex","adaptive immune response","cytokine receptor binding","response to lipopolysaccharide","positive regulation of immune response","cellular response to interleukin-1","response to lipid","chemokine-mediated signaling pathway","regulation of leukocyte activation","cell-cell adhesion","regulation of lymphocyte activation","regulation of T cell activation","cellular response to lipopolysaccharide","signaling receptor activity","immune response-regulating cell surface receptor signaling pathway","MHC protein complex","plasma membrane signaling receptor complex","activation of immune response","positive regulation of cell-cell adhesion","leukocyte chemotaxis","immune response-activating signaling pathway","neutrophil chemotaxis","lymphocyte mediated immunity","postsynaptic specialization membrane","leukocyte mediated immunity","antigen processing and presentation of exogenous peptide antigen via MHC class II","positive regulation of T cell activation","immune response-activating cell surface receptor signaling pathway","protein tyrosine kinase activator activity","T cell activation","positive regulation of response to external stimulus","positive regulation of lymphocyte activation","transmembrane signaling receptor activity","cellular response to lipid","regulation of homotypic cell-cell adhesion","multivesicular body","mononuclear cell differentiation","antigen binding","positive regulation of chemokine production","regulation of leukocyte cell-cell adhesion","regulation of ERK1 and ERK2 cascade","regulation of MAPK cascade","lipopolysaccharide-mediated signaling pathway"],"Count":[19,18,16,12,8,10,11,5,15,14,13,4,9,14,9,11,16,7,15,6,15,15,14,12,9,11,10,4,6,12,10,8,11,6,10,4,11,4,9,9,3,13,12,10,8,11,4,4,12,4,5,10,8,12,5],"% DE":[0.27536231884058,0.257142857142857,0.231884057971014,0.173913043478261,0.115942028985507,0.142857142857143,0.159420289855072,0.072463768115942,0.214285714285714,0.2,0.188405797101449,0.0579710144927536,0.130434782608696,0.2,0.130434782608696,0.157142857142857,0.228571428571429,0.1,0.214285714285714,0.0857142857142857,0.214285714285714,0.214285714285714,0.2,0.171428571428571,0.128571428571429,0.159420289855072,0.142857142857143,0.0579710144927536,0.0869565217391304,0.171428571428571,0.142857142857143,0.114285714285714,0.157142857142857,0.0857142857142857,0.142857142857143,0.0579710144927536,0.157142857142857,0.0571428571428571,0.128571428571429,0.128571428571429,0.0434782608695652,0.185714285714286,0.171428571428571,0.142857142857143,0.115942028985507,0.157142857142857,0.0571428571428571,0.0579710144927536,0.171428571428571,0.0579710144927536,0.0714285714285714,0.142857142857143,0.114285714285714,0.171428571428571,0.0714285714285714],"% BG":[0.0453933050057001,0.0440330509360946,0.0403150585552907,0.0218675510415587,0.00720852486418721,0.0114004811212216,0.0220748264068815,0.00198495612202257,0.0384896977303629,0.0333647108043092,0.0353404497875427,0.00104471374843293,0.0150274639859053,0.0338876686539065,0.0144170497283744,0.0193494404351009,0.0474845727434369,0.00585712791549001,0.0430917268068194,0.00386988808702019,0.0440330509360946,0.0442422340759335,0.0385942893002824,0.0283443154481749,0.0142244535090472,0.0300877559548684,0.0196632151448593,0.00217639133588973,0.0086019276608975,0.0328417529547119,0.0226963706725238,0.013283129379772,0.0283443154481749,0.0061709026252484,0.0231147369522017,0.00310913047984247,0.0290764564376111,0.0017780566886309,0.0189310741554231,0.0192448488651815,0.00125365649811952,0.0442422340759335,0.0376529651710072,0.0255203430603493,0.0207898035938153,0.0317958372555172,0.00219642296830875,0.00414550730645663,0.0392218387197992,0.00376096949435855,0.0047066206463759,0.0274029913188997,0.0168392427570338,0.0411044869783495,0.0050203953561343],"p.adjust":[1.7870208489158e-08,1.38530209917904e-06,1.47919131574344e-06,3.94496661638645e-06,6.71993301414294e-06,1.0020222506245e-05,4.2563204623082e-05,4.28163722333418e-05,8.68448119262888e-05,0.000104257026268312,0.000105447992688785,0.000119127083562014,0.00012277054402761,0.000126604511722633,0.000133653946599791,0.000155213840603238,0.000217763665611408,0.00026514515209427,0.000381604266816312,0.000440797352179053,0.000504655438135076,0.000536467914886948,0.000628960116632342,0.000949339566250551,0.00104411132396199,0.00138258230606473,0.00177611406794809,0.00200081089867258,0.00401632141455187,0.00449844271357777,0.0065178055293799,0.00673487076475281,0.00695801350520902,0.00752809700460177,0.00767480289730777,0.00872781008387083,0.0088939017546638,0.0106857405051908,0.0112847379922364,0.0129017418476059,0.0178435654785895,0.0179597220018572,0.0183228167954308,0.0184444076083682,0.0202689101679896,0.020829901047713,0.0262838086660255,0.0275803800916592,0.0276439468222411,0.0290116942770854,0.0324402860862723,0.0343081326243955,0.0387132049355074,0.0441165899526625,0.0446989829816067],"geneID":["Fam20a; Areg; Lgals7; Cst7; Ccl5; Cd74; Cd40; Cxcl2; Serpinf1; Gpld1; Il1b; Il1rn; Slpi; Cd14; Cxcl16; Chil5; Ccl8; Klk1b11; Ccl7","P2rx7; Acod1; Cst7; Havcr2; Ccl5; Cd40; Lyn; Cxcl2; Serpinf1; Il1b; Il1rn; Cd14; Chil5; Pld4; Ccl8; Ifngr2; Ccr4; Ccl7","Areg; P2rx7; Lag3; Plaur; Havcr2; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Ocln; Cd14; Cd79a; Ccr4","P2rx7; Lag3; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4","Areg; Ccl5; Cxcl2; Il1b; Il1rn; Cxcl16; Ccl8; Ccl7","Acod1; Ccl5; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cxcl16; Ccl8; Ccl7","Trat1; Cacna1s; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd79a; Cnih2","Ccl5; Cxcl2; Cxcl16; Ccl8; Ccl7","Fam20a; P2rx7; Acod1; Havcr2; Bnip3; Ccl5; Cd79b; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16; Ociad2; Tnip3","Tespa1; Lag3; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; Sh2b3","P2rx7; Lag3; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd14; Cd79a; Ccr4","H2-Ab1; Cd74; H2-Aa; H2-Eb1","Trat1; Cd79b; Cd74; Cd40; Lyn; Gpr160; Cd14; Cd79a; Cnih2","P2rx7; Lag3; Trat1; Havcr2; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b; Cd79a","Ccl5; Lyn; Cxcl2; Il1b; Il1rn; Cxcl16; Sh2b3; Ccl8; Ccl7","P2rx7; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16; Tnip3","Tespa1; P2rx7; Acod1; Lag3; Trat1; Havcr2; Cd79b; H2-Ab1; Cd74; Cd40; Lyn; Gpld1; Il1b; Cd14; Cd79a; Tnip3","Acod1; Ccl5; Cd40; Il1b; Il1rn; Ccl8; Ccl7","Id3; Areg; P2rx7; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Gpld1; Il1b; Il1rn; Slpi; Cd14; Cxcl16; Tnip3","Ccl5; Lyn; Cxcl2; Sh2b3; Ccl8; Ccl7","Tespa1; Cst7; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3","Tespa1; Lgals7; Lag3; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; Sh2b3","Tespa1; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3","Tespa1; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3","Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Il1b; Cd14; Cxcl16; Tnip3","P2rx7; Lag3; Plaur; Cd79b; Cd74; Cd40; H2-Eb1; Gpr160; Cd14; Cd79a; Ccr4","Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Gpld1; Cd14; Cd79a; Tnip3","H2-Ab1; Cd74; H2-Aa; H2-Eb1","Trat1; Cd79b; Cd40; Lyn; Cd79a; Cnih2","Tespa1; Acod1; Trat1; Havcr2; Cd79b; Cd40; Lyn; Gpld1; Il1b; Cd14; Cd79a; Tnip3","Tespa1; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b","Ccl5; Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccl7","Tespa1; Acod1; Trat1; Havcr2; Cd79b; Cd40; Lyn; Gpld1; Cd14; Cd79a; Tnip3","Ccl5; Cd74; Cxcl2; Il1b; Ccl8; Ccl7","P2rx7; Lag3; Havcr2; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Gzmm; Il1b","Rapsn; Slc16a3; Dlg2; Cnih2","P2rx7; Lag3; Havcr2; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b","H2-Ab1; Cd74; H2-Aa; H2-Eb1","Tespa1; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b","Tespa1; Acod1; Trat1; Cd79b; Lyn; Gpld1; Cd14; Cd79a; Tnip3","Ccl5; Cd79b; Cd79a","Tespa1; P2rx7; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3","Acod1; Lag3; Havcr2; Ccl5; Cd74; Cd40; Lyn; Il1b; Cd14; Ccr4; Tnip3; Ccl7","Tespa1; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b","P2rx7; Lag3; Cd79b; Cd74; H2-Eb1; Gpr160; Cd79a; Ccr4","Id3; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Gpld1; Il1b; Cd14; Cxcl16; Tnip3","Plaur; Ccl5; Lyn; Sh2b3","Cst7; H2-Ab1; Cd74; Cd79a","Tespa1; Lag3; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Lyn; Il1b; Sh2b3; Cd79a","H2-Ab1; H2-Aa; Cd40; H2-Eb1","Havcr2; Ccl5; Cd74; Il1b; Chil5","Tespa1; Lag3; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b","Havcr2; Atf3; Ccl5; Cd74; Lyn; Il1b; Ccl8; Ccl7","P2rx7; Havcr2; Atf3; Ccl5; Cd74; Cd40; Lyn; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7","Acod1; Ccl5; Lyn; Il1b; Cd14"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["1.79e-08","1.39e-06","1.48e-06","3.94e-06","6.72e-06","1.00e-05","4.26e-05","4.28e-05","8.68e-05","1.04e-04","1.05e-04","1.19e-04","1.23e-04","1.27e-04","1.34e-04","1.55e-04","2.18e-04","2.65e-04","3.82e-04","4.41e-04","5.05e-04","5.36e-04","6.29e-04","9.49e-04","0.001","0.001","0.002","0.002","0.004","0.004","0.007","0.007","0.007","0.008","0.008","0.009","0.009","0.011","0.011","0.013","0.018","0.018","0.018","0.018","0.020","0.021","0.026","0.028","0.028","0.029","0.032","0.034","0.039","0.044","0.045"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Fam20a; Areg; Lgals7; Cst7; Ccl5; Cd74; Cd40; Cxcl2; Serpinf1; Gpld1; Il1b; Il1rn; Slpi; Cd14; Cxcl16; Chil5; Ccl8; Klk1b11; Ccl7"},"children":["Fam20a; Areg; Lgals7; Cst7; Ccl5; Cd74; Cd40; Cxcl2; Serpinf1; Gpld1; Il1b; Il1rn; Slpi; Cd14; Cx..."]},{"name":"span","attribs":{"title":"P2rx7; Acod1; Cst7; Havcr2; Ccl5; Cd40; Lyn; Cxcl2; Serpinf1; Il1b; Il1rn; Cd14; Chil5; Pld4; Ccl8; Ifngr2; Ccr4; Ccl7"},"children":["P2rx7; Acod1; Cst7; Havcr2; Ccl5; Cd40; Lyn; Cxcl2; Serpinf1; Il1b; Il1rn; Cd14; Chil5; Pld4; Ccl..."]},{"name":"span","attribs":{"title":"Areg; P2rx7; Lag3; Plaur; Havcr2; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Ocln; Cd14; Cd79a; Ccr4"},"children":["Areg; P2rx7; Lag3; Plaur; Havcr2; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Ocln; Cd14; Cd7..."]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4"},"children":["P2rx7; Lag3; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"Areg; Ccl5; Cxcl2; Il1b; Il1rn; Cxcl16; Ccl8; Ccl7"},"children":["Areg; Ccl5; Cxcl2; Il1b; Il1rn; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Acod1; Ccl5; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cxcl16; Ccl8; Ccl7"},"children":["Acod1; Ccl5; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Trat1; Cacna1s; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd79a; Cnih2"},"children":["Trat1; Cacna1s; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd79a; Cnih2"]},{"name":"span","attribs":{"title":"Ccl5; Cxcl2; Cxcl16; Ccl8; Ccl7"},"children":["Ccl5; Cxcl2; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Fam20a; P2rx7; Acod1; Havcr2; Bnip3; Ccl5; Cd79b; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16; Ociad2; Tnip3"},"children":["Fam20a; P2rx7; Acod1; Havcr2; Bnip3; Ccl5; Cd79b; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16; Ociad2; T..."]},{"name":"span","attribs":{"title":"Tespa1; Lag3; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; Sh2b3"},"children":["Tespa1; Lag3; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; Sh2b3"]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd14; Cd79a; Ccr4"},"children":["P2rx7; Lag3; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd14; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"H2-Ab1; Cd74; H2-Aa; H2-Eb1"},"children":["H2-Ab1; Cd74; H2-Aa; H2-Eb1"]},{"name":"span","attribs":{"title":"Trat1; Cd79b; Cd74; Cd40; Lyn; Gpr160; Cd14; Cd79a; Cnih2"},"children":["Trat1; Cd79b; Cd74; Cd40; Lyn; Gpr160; Cd14; Cd79a; Cnih2"]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Trat1; Havcr2; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b; Cd79a"},"children":["P2rx7; Lag3; Trat1; Havcr2; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b; Cd79a"]},{"name":"span","attribs":{"title":"Ccl5; Lyn; Cxcl2; Il1b; Il1rn; Cxcl16; Sh2b3; Ccl8; Ccl7"},"children":["Ccl5; Lyn; Cxcl2; Il1b; Il1rn; Cxcl16; Sh2b3; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"P2rx7; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16; Tnip3"},"children":["P2rx7; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16; Tnip3"]},{"name":"span","attribs":{"title":"Tespa1; P2rx7; Acod1; Lag3; Trat1; Havcr2; Cd79b; H2-Ab1; Cd74; Cd40; Lyn; Gpld1; Il1b; Cd14; Cd79a; Tnip3"},"children":["Tespa1; P2rx7; Acod1; Lag3; Trat1; Havcr2; Cd79b; H2-Ab1; Cd74; Cd40; Lyn; Gpld1; Il1b; Cd14; Cd7..."]},{"name":"span","attribs":{"title":"Acod1; Ccl5; Cd40; Il1b; Il1rn; Ccl8; Ccl7"},"children":["Acod1; Ccl5; Cd40; Il1b; Il1rn; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Id3; Areg; P2rx7; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Gpld1; Il1b; Il1rn; Slpi; Cd14; Cxcl16; Tnip3"},"children":["Id3; Areg; P2rx7; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Gpld1; Il1b; Il1rn; Slpi; Cd14; Cxcl16; Tnip3"]},{"name":"span","attribs":{"title":"Ccl5; Lyn; Cxcl2; Sh2b3; Ccl8; Ccl7"},"children":["Ccl5; Lyn; Cxcl2; Sh2b3; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Tespa1; Cst7; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3"},"children":["Tespa1; Cst7; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3"]},{"name":"span","attribs":{"title":"Tespa1; Lgals7; Lag3; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; Sh2b3"},"children":["Tespa1; Lgals7; Lag3; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; S..."]},{"name":"span","attribs":{"title":"Tespa1; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3"},"children":["Tespa1; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3"]},{"name":"span","attribs":{"title":"Tespa1; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3"},"children":["Tespa1; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3"]},{"name":"span","attribs":{"title":"Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Il1b; Cd14; Cxcl16; Tnip3"},"children":["Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Il1b; Cd14; Cxcl16; Tnip3"]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Plaur; Cd79b; Cd74; Cd40; H2-Eb1; Gpr160; Cd14; Cd79a; Ccr4"},"children":["P2rx7; Lag3; Plaur; Cd79b; Cd74; Cd40; H2-Eb1; Gpr160; Cd14; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Gpld1; Cd14; Cd79a; Tnip3"},"children":["Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Gpld1; Cd14; Cd79a; Tnip3"]},{"name":"span","attribs":{"title":"H2-Ab1; Cd74; H2-Aa; H2-Eb1"},"children":["H2-Ab1; Cd74; H2-Aa; H2-Eb1"]},{"name":"span","attribs":{"title":"Trat1; Cd79b; Cd40; Lyn; Cd79a; Cnih2"},"children":["Trat1; Cd79b; Cd40; Lyn; Cd79a; Cnih2"]},{"name":"span","attribs":{"title":"Tespa1; Acod1; Trat1; Havcr2; Cd79b; Cd40; Lyn; Gpld1; Il1b; Cd14; Cd79a; Tnip3"},"children":["Tespa1; Acod1; Trat1; Havcr2; Cd79b; Cd40; Lyn; Gpld1; Il1b; Cd14; Cd79a; Tnip3"]},{"name":"span","attribs":{"title":"Tespa1; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"},"children":["Tespa1; Plaur; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"]},{"name":"span","attribs":{"title":"Ccl5; Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccl7"},"children":["Ccl5; Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Tespa1; Acod1; Trat1; Havcr2; Cd79b; Cd40; Lyn; Gpld1; Cd14; Cd79a; Tnip3"},"children":["Tespa1; Acod1; Trat1; Havcr2; Cd79b; Cd40; Lyn; Gpld1; Cd14; Cd79a; Tnip3"]},{"name":"span","attribs":{"title":"Ccl5; Cd74; Cxcl2; Il1b; Ccl8; Ccl7"},"children":["Ccl5; Cd74; Cxcl2; Il1b; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Havcr2; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Gzmm; Il1b"},"children":["P2rx7; Lag3; Havcr2; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Gzmm; Il1b"]},{"name":"span","attribs":{"title":"Rapsn; Slc16a3; Dlg2; Cnih2"},"children":["Rapsn; Slc16a3; Dlg2; Cnih2"]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Havcr2; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b"},"children":["P2rx7; Lag3; Havcr2; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b"]},{"name":"span","attribs":{"title":"H2-Ab1; Cd74; H2-Aa; H2-Eb1"},"children":["H2-Ab1; Cd74; H2-Aa; H2-Eb1"]},{"name":"span","attribs":{"title":"Tespa1; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"},"children":["Tespa1; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"]},{"name":"span","attribs":{"title":"Tespa1; Acod1; Trat1; Cd79b; Lyn; Gpld1; Cd14; Cd79a; Tnip3"},"children":["Tespa1; Acod1; Trat1; Cd79b; Lyn; Gpld1; Cd14; Cd79a; Tnip3"]},{"name":"span","attribs":{"title":"Ccl5; Cd79b; Cd79a"},"children":["Ccl5; Cd79b; Cd79a"]},{"name":"span","attribs":{"title":"Tespa1; P2rx7; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3"},"children":["Tespa1; P2rx7; Lag3; Havcr2; Ccl5; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3"]},{"name":"span","attribs":{"title":"Acod1; Lag3; Havcr2; Ccl5; Cd74; Cd40; Lyn; Il1b; Cd14; Ccr4; Tnip3; Ccl7"},"children":["Acod1; Lag3; Havcr2; Ccl5; Cd74; Cd40; Lyn; Il1b; Cd14; Ccr4; Tnip3; Ccl7"]},{"name":"span","attribs":{"title":"Tespa1; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b"},"children":["Tespa1; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b"]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Cd79b; Cd74; H2-Eb1; Gpr160; Cd79a; Ccr4"},"children":["P2rx7; Lag3; Cd79b; Cd74; H2-Eb1; Gpr160; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"Id3; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Gpld1; Il1b; Cd14; Cxcl16; Tnip3"},"children":["Id3; Acod1; Havcr2; Ccl5; Lyn; Cxcl2; Gpld1; Il1b; Cd14; Cxcl16; Tnip3"]},{"name":"span","attribs":{"title":"Plaur; Ccl5; Lyn; Sh2b3"},"children":["Plaur; Ccl5; Lyn; Sh2b3"]},{"name":"span","attribs":{"title":"Cst7; H2-Ab1; Cd74; Cd79a"},"children":["Cst7; H2-Ab1; Cd74; Cd79a"]},{"name":"span","attribs":{"title":"Tespa1; Lag3; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Lyn; Il1b; Sh2b3; Cd79a"},"children":["Tespa1; Lag3; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Lyn; Il1b; Sh2b3; Cd79a"]},{"name":"span","attribs":{"title":"H2-Ab1; H2-Aa; Cd40; H2-Eb1"},"children":["H2-Ab1; H2-Aa; Cd40; H2-Eb1"]},{"name":"span","attribs":{"title":"Havcr2; Ccl5; Cd74; Il1b; Chil5"},"children":["Havcr2; Ccl5; Cd74; Il1b; Chil5"]},{"name":"span","attribs":{"title":"Tespa1; Lag3; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"},"children":["Tespa1; Lag3; Havcr2; Ccl5; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"]},{"name":"span","attribs":{"title":"Havcr2; Atf3; Ccl5; Cd74; Lyn; Il1b; Ccl8; Ccl7"},"children":["Havcr2; Atf3; Ccl5; Cd74; Lyn; Il1b; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"P2rx7; Havcr2; Atf3; Ccl5; Cd74; Cd40; Lyn; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7"},"children":["P2rx7; Havcr2; Atf3; Ccl5; Cd74; Cd40; Lyn; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Acod1; Ccl5; Lyn; Il1b; Cd14"},"children":["Acod1; Ccl5; Lyn; Il1b; Cd14"]}]}],"filterable":true,"dataKey":"9852e16eb6d3ba68641cc086da6a7a9b"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="DNvDP" class="section level3">
<h3 class="tabset">DNvDP</h3>
<em>All 3 ontologies considered enriched in the set of 53 DE genes mapped to EntrezIDs from the comparison DNvDP. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-6974f81ac9bb9e59de3c" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-6974f81ac9bb9e59de3c">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["MF","MF","CC"],"ID":["GO:0038023","GO:0004888","GO:0009986"],"Description":["signaling receptor activity","transmembrane signaling receptor activity","cell surface"],"Count":[10,7,9],"% DE":[0.2,0.14,0.173076923076923],"% BG":[0.0300877559548684,0.0207898035938153,0.0403150585552907],"p.adjust":[0.000292674501289461,0.0112456457961877,0.0262981476712171],"geneID":["Cxcr5; Il12rb2; Tnfrsf9; Klrd1; Gpr15; Cd7; Lag3; Gpr160; Plaur; Itga2","Cxcr5; Il12rb2; Klrd1; Gpr15; Lag3; Gpr160; Itga2","Cxcr5; Il12rb2; Havcr2; Sell; Klrd1; Lag3; Ocln; Plaur; Itga2"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["2.93e-04","0.011","0.026"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Cxcr5; Il12rb2; Tnfrsf9; Klrd1; Gpr15; Cd7; Lag3; Gpr160; Plaur; Itga2"},"children":["Cxcr5; Il12rb2; Tnfrsf9; Klrd1; Gpr15; Cd7; Lag3; Gpr160; Plaur; Itga2"]},{"name":"span","attribs":{"title":"Cxcr5; Il12rb2; Klrd1; Gpr15; Lag3; Gpr160; Itga2"},"children":["Cxcr5; Il12rb2; Klrd1; Gpr15; Lag3; Gpr160; Itga2"]},{"name":"span","attribs":{"title":"Cxcr5; Il12rb2; Havcr2; Sell; Klrd1; Lag3; Ocln; Plaur; Itga2"},"children":["Cxcr5; Il12rb2; Havcr2; Sell; Klrd1; Lag3; Ocln; Plaur; Itga2"]}]}],"filterable":true,"dataKey":"103d27d4fa7fcd34b4b7e89f061c516c"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="DNvCD49b" class="section level3">
<h3 class="tabset">DNvCD49b</h3>
<em>All 10 ontologies considered enriched in the set of 16 DE genes mapped to EntrezIDs from the comparison DNvCD49b. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-0a6d22f440e562ee4442" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0a6d22f440e562ee4442">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["BP","BP","MF","CC","BP","BP","MF","BP","CC","BP"],"ID":["GO:0050900","GO:0002523","GO:0005504","GO:0009897","GO:1903034","GO:0030193","GO:0036041","GO:1903036","GO:0005615","GO:0048708"],"Description":["leukocyte migration","leukocyte migration involved in inflammatory response","fatty acid binding","external side of plasma membrane","regulation of response to wounding","regulation of blood coagulation","long-chain fatty acid binding","positive regulation of response to wounding","extracellular space","astrocyte differentiation"],"Count":[6,3,3,5,4,3,2,3,5,3],"% DE":[0.428571428571429,0.214285714285714,0.2,0.333333333333333,0.285714285714286,0.214285714285714,0.133333333333333,0.214285714285714,0.333333333333333,0.214285714285714],"% BG":[0.0230101453822822,0.00125509883903357,0.00250731299623903,0.0218675510415587,0.0079489593138793,0.00251019767806715,0.00104471374843293,0.00376529651710072,0.0453933050057001,0.00491580378621483],"p.adjust":[0.000274146877965061,0.00041907070011349,0.00053757217462362,0.00069479293731481,0.00266839232715162,0.00381569055249973,0.00890930059574365,0.0133216336099667,0.0224712646989465,0.0299672529795065],"geneID":["S100a8; Cd24a; Cxcr5; S100a9; Gpr15; Cd9","S100a8; Cd24a; S100a9","S100a8; Id3; Cd36","Cd24a; Cxcr5; Cd9; Cd36; Cd8a","Cd24a; S100a9; Cd9; Cd36","S100a9; Cd9; Cd36","S100a8; Cd36","Cd24a; S100a9; Cd36","S100a8; Igfbp7; S100a9; Cd9; Ggt1","S100a8; Eomes; S100a9"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["2.74e-04","4.19e-04","5.38e-04","6.95e-04","0.003","0.004","0.009","0.013","0.022","0.030"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"S100a8; Cd24a; Cxcr5; S100a9; Gpr15; Cd9"},"children":["S100a8; Cd24a; Cxcr5; S100a9; Gpr15; Cd9"]},{"name":"span","attribs":{"title":"S100a8; Cd24a; S100a9"},"children":["S100a8; Cd24a; S100a9"]},{"name":"span","attribs":{"title":"S100a8; Id3; Cd36"},"children":["S100a8; Id3; Cd36"]},{"name":"span","attribs":{"title":"Cd24a; Cxcr5; Cd9; Cd36; Cd8a"},"children":["Cd24a; Cxcr5; Cd9; Cd36; Cd8a"]},{"name":"span","attribs":{"title":"Cd24a; S100a9; Cd9; Cd36"},"children":["Cd24a; S100a9; Cd9; Cd36"]},{"name":"span","attribs":{"title":"S100a9; Cd9; Cd36"},"children":["S100a9; Cd9; Cd36"]},{"name":"span","attribs":{"title":"S100a8; Cd36"},"children":["S100a8; Cd36"]},{"name":"span","attribs":{"title":"Cd24a; S100a9; Cd36"},"children":["Cd24a; S100a9; Cd36"]},{"name":"span","attribs":{"title":"S100a8; Igfbp7; S100a9; Cd9; Ggt1"},"children":["S100a8; Igfbp7; S100a9; Cd9; Ggt1"]},{"name":"span","attribs":{"title":"S100a8; Eomes; S100a9"},"children":["S100a8; Eomes; S100a9"]}]}],"filterable":true,"dataKey":"83c2dcf5b28f882039a46c4a58f8a8a7"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="LAG3vCD49b" class="section level3">
<h3 class="tabset">LAG3vCD49b</h3>
<em>All 36 ontologies considered enriched in the set of 151 DE genes mapped to EntrezIDs from the comparison LAG3vCD49b. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-937dc4029594cb19c8de" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-937dc4029594cb19c8de">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["CC","CC","CC","MF","BP","BP","CC","MF","BP","MF","BP","BP","MF","CC","MF","MF","MF","BP","BP","MF","BP","BP","BP","MF","BP","BP","BP","BP","BP","MF","BP","BP","BP","BP","BP","BP"],"ID":["GO:0005615","GO:0009986","GO:0009897","GO:0038023","GO:0006935","GO:0098609","GO:0098552","GO:0004888","GO:0050900","GO:0030545","GO:0006954","GO:0030595","GO:0140375","GO:0043235","GO:0050839","GO:0005125","GO:0048018","GO:0031295","GO:0030155","GO:0004866","GO:0007186","GO:0002252","GO:0002449","GO:0001968","GO:0022408","GO:0048247","GO:0002418","GO:0050731","GO:0032103","GO:0019955","GO:0050730","GO:0018108","GO:0050865","GO:0050866","GO:0071621","GO:0072676"],"Description":["extracellular space","cell surface","external side of plasma membrane","signaling receptor activity","chemotaxis","cell-cell adhesion","side of membrane","transmembrane signaling receptor activity","leukocyte migration","signaling receptor regulator activity","inflammatory response","leukocyte chemotaxis","immune receptor activity","receptor complex","cell adhesion molecule binding","cytokine activity","receptor ligand activity","T cell costimulation","regulation of cell adhesion","endopeptidase inhibitor activity","G protein-coupled receptor signaling pathway","immune effector process","lymphocyte mediated immunity","fibronectin binding","negative regulation of cell-cell adhesion","lymphocyte chemotaxis","immune response to tumor cell","positive regulation of peptidyl-tyrosine phosphorylation","positive regulation of response to external stimulus","cytokine binding","regulation of peptidyl-tyrosine phosphorylation","peptidyl-tyrosine phosphorylation","regulation of cell activation","negative regulation of cell activation","granulocyte chemotaxis","lymphocyte migration"],"Count":[36,27,20,21,20,26,22,17,18,12,22,12,8,11,10,8,10,6,22,7,15,20,14,4,11,6,5,9,18,7,11,12,20,11,8,8],"% DE":[0.243243243243243,0.182432432432432,0.135135135135135,0.143835616438356,0.137931034482759,0.179310344827586,0.148648648648649,0.116438356164384,0.124137931034483,0.0821917808219178,0.151724137931034,0.0827586206896552,0.0547945205479452,0.0743243243243243,0.0684931506849315,0.0547945205479452,0.0684931506849315,0.0413793103448276,0.151724137931034,0.0479452054794521,0.103448275862069,0.137931034482759,0.096551724137931,0.0273972602739726,0.0758620689655172,0.0413793103448276,0.0344827586206897,0.0620689655172414,0.124137931034483,0.0479452054794521,0.0758620689655172,0.0827586206896552,0.137931034482759,0.0758620689655172,0.0551724137931034,0.0551724137931034],"% BG":[0.0453933050057001,0.0403150585552907,0.0218675510415587,0.0300877559548684,0.0257295262001883,0.0442422340759335,0.0353404497875427,0.0207898035938153,0.0230101453822822,0.0135812787296281,0.0440330509360946,0.013283129379772,0.00679063936481404,0.0150274639859053,0.0118052653572921,0.00720852486418721,0.0120142081069787,0.00261478924798661,0.0484258968727121,0.00605933974091099,0.0255203430603493,0.0438238677962556,0.0231147369522017,0.0014625992478061,0.0143290450789666,0.00334693023742286,0.00198723982846983,0.00930864972283234,0.0376529651710072,0.00679063936481404,0.014852002928564,0.0178851584562284,0.0462294739044033,0.0152703692082418,0.00763518460412091,0.00773977617404037],"p.adjust":[7.57875963346183e-15,6.39438503968382e-09,1.26134210622078e-08,8.34938694421358e-07,1.95152087189775e-06,2.06739945169291e-06,2.07073467950921e-06,3.0354019557398e-06,1.33833245662573e-05,0.000233612954001953,0.000859668647741463,0.0011773722879551,0.00220141845663167,0.00282576045456635,0.00305760927658355,0.00346417442103898,0.00357675779245336,0.00398274978890464,0.00428616927075009,0.00981045628034525,0.0106391403076686,0.0125261914684115,0.0160543581995306,0.0173591709240254,0.0179326169846611,0.0186720035737218,0.0190736323519492,0.0195121556042834,0.0206903642837498,0.0207818808218861,0.0253088307969164,0.0272265474815945,0.0277889427204841,0.032988559560672,0.0350754082842243,0.0388282815293328],"geneID":["Lgals7; Igfbp7; Areg; Cst7; Angptl2; Serpinf1; Fam20a; Twsg1; Gcg; Ccl3; Ccl5; Sorl1; Cd9; Nrn1; Il10; Cpm; Ccl4; S100a8; Prf1; Cd109; Serpine2; Pla1a; Klk1b9; Ggt1; Bmp7; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Car4; Ear2; Klk1b11; Prrg4; Cd40; Cd74","P2rx7; Areg; Lilrb4b; Lag3; Itgb3; Lilrb4a; Sorl1; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr5; Epcam; Itga3; Cd36; Sell; Cd80; Ocln; Cd8a; Car4; Klrk1; Ptprk; Cd40; Ephb6; Cd74","P2rx7; Lag3; Itgb3; Lilrb4a; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr5; Itga3; Cd36; Sell; Cd80; Cd8a; Car4; Klrk1; Cd40; Cd74","P2rx7; Lilrb4b; Lag3; Lilrb4a; Sorl1; Itga2; Ccr4; Il1r2; Ptch1; Gpr15; Cxcr5; Ddr1; Cd36; Cd80; Vipr1; Ccr3; Klrk1; Reck; Cd40; Ephb6; Cd74","Oxsr1; Itgb3; Ccl3; Ccl5; Itga2; Ccl4; S100a8; Ccr4; Ptch1; Egr2; Cxcr5; Bmp7; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Ear2; Ephb6; Cd74","Lgals7; Perp; Lilrb4b; Tespa1; Lag3; Twsg1; Itgb3; Dsp; Ccl5; Lilrb4a; Cd9; Itga2; Il10; S100a8; Serpine2; Cd24a; Epcam; Bmp7; Pkp4; Itga3; Tnfsf14; Sell; Cd80; S100a9; Ephb6; Cd74","P2rx7; Lag3; Itgb3; Lilrb4a; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr5; Pkp4; Itga3; Cd36; Sell; Cd80; Cd8a; Car4; S100a6; Klrk1; Cd40; Cd74","P2rx7; Lilrb4b; Lag3; Lilrb4a; Sorl1; Itga2; Ccr4; Il1r2; Ptch1; Gpr15; Cxcr5; Ddr1; Vipr1; Ccr3; Klrk1; Ephb6; Cd74","Oxsr1; Itgb3; Ccl3; Ccl5; Cd9; Itga2; Ccl4; S100a8; Cd24a; Gpr15; Cxcr5; Itga3; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Cd74","Areg; Lilrb4b; Gcg; Ccl3; Ccl5; Lilrb4a; Il10; Ccl4; Insl6; Bmp7; Tnfsf14; Cxcl16","P2rx7; Cst7; Lilrb4b; Serpinf1; Ccl3; Ccl5; Lilrb4a; Itga2; Il10; Nkg7; Ccl4; S100a8; Ccr4; Il1r2; Dnase1l3; Cd24a; Ggt1; S100a9; Ccr3; Ifngr2; Cd40; Ephb6","Oxsr1; Ccl3; Ccl5; Ccl4; S100a8; Cxcr5; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Cd74","Lilrb4b; Lilrb4a; Ccr4; Il1r2; Cxcr5; Ccr3; Klrk1; Cd74","Trat1; Itgb3; Itga2; Nrn1; Ddr1; Itga3; Cd36; Cd8a; Vipr1; Cd40; Cd74","Itgb3; Dsp; Lilrb4a; Cd9; Itga2; Cd226; Epcam; Pkp4; Itga3; Sell","Areg; Ccl3; Ccl5; Il10; Ccl4; Bmp7; Tnfsf14; Cxcl16","Areg; Gcg; Ccl3; Ccl5; Il10; Ccl4; Insl6; Bmp7; Tnfsf14; Cxcl16","Lilrb4b; Lilrb4a; Cd24a; Tnfsf14; Cd80; Ephb6","Lilrb4b; Tespa1; Lag3; Twsg1; Itgb3; Ccl5; Lilrb4a; Cd9; Itga2; Il10; Cass4; Serpine2; Cd24a; Epcam; Bmp7; Ddr1; Itga3; Cd36; Tnfsf14; Cd80; Ephb6; Cd74","Cst7; Serpinf1; Cd109; Serpine2; Tnfsf14; Arrb1; Reck","Areg; Gcg; Itgb3; Ccl3; Ccl5; Sorl1; Dgkg; Ccl4; Ccr4; Gpr15; Slc39a14; Cxcr5; Vipr1; Ccr3; Arrb1","P2rx7; Lilrb4b; Lag3; Lilrb4a; Eomes; Il10; Nkg7; Prf1; Dnase1l3; Cd24a; Cd226; Gzmm; Cd36; Cgas; Cd8a; Klrk1; Zpbp2; Cd40; Ephb6; Cd74","P2rx7; Lilrb4b; Lag3; Lilrb4a; Nkg7; Prf1; Cd24a; Cd226; Gzmm; Cd8a; Klrk1; Cd40; Ephb6; Cd74","Lilrb4b; Itgb3; Lilrb4a; Itga3","Lilrb4b; Lag3; Twsg1; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Epcam; Cd80; Cd74","Oxsr1; Ccl3; Ccl5; Ccl4; Tnfsf14; Cxcl16","Nkg7; Prf1; Cd24a; Cd226; Klrk1","Areg; Itgb3; Ccl5; Cass4; Cd24a; Cd36; Cd80; Cd40; Cd74","Oxsr1; Lag3; Ccl3; Ccl5; Itga2; Nkg7; S100a8; Ccr4; Cd24a; Cd226; Cd36; Tnfsf14; Cgas; Sell; S100a9; Klrk1; Cd40; Cd74","Twsg1; Itgb3; Il1r2; Cd109; Cd36; Ccr3; Cd74","Areg; Lilrb4b; Itgb3; Ccl5; Lilrb4a; Cass4; Cd24a; Cd36; Cd80; Cd40; Cd74","Areg; Lilrb4b; Itgb3; Ccl5; Lilrb4a; Cass4; Cd24a; Ddr1; Cd36; Cd80; Cd40; Cd74","Cst7; Lilrb4b; Tespa1; Tox; Lag3; Twsg1; Ccl5; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Cd226; Tnfsf14; Cgas; Cd80; Klrk1; Cd40; Ephb6; Cd74","Cst7; Lilrb4b; Lag3; Twsg1; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Cd80; Cd74","Ccl3; Ccl5; Ccl4; S100a8; Sell; S100a9; Ccr3; Cd74","Oxsr1; Itgb3; Ccl3; Ccl5; Ccl4; Gpr15; Tnfsf14; Cxcl16"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["7.58e-15","6.39e-09","1.26e-08","8.35e-07","1.95e-06","2.07e-06","2.07e-06","3.04e-06","1.34e-05","2.34e-04","8.60e-04","0.001","0.002","0.003","0.003","0.003","0.004","0.004","0.004","0.010","0.011","0.013","0.016","0.017","0.018","0.019","0.019","0.020","0.021","0.021","0.025","0.027","0.028","0.033","0.035","0.039"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Lgals7; Igfbp7; Areg; Cst7; Angptl2; Serpinf1; Fam20a; Twsg1; Gcg; Ccl3; Ccl5; Sorl1; Cd9; Nrn1; Il10; Cpm; Ccl4; S100a8; Prf1; Cd109; Serpine2; Pla1a; Klk1b9; Ggt1; Bmp7; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Car4; Ear2; Klk1b11; Prrg4; Cd40; Cd74"},"children":["Lgals7; Igfbp7; Areg; Cst7; Angptl2; Serpinf1; Fam20a; Twsg1; Gcg; Ccl3; Ccl5; Sorl1; Cd9; Nrn1; ..."]},{"name":"span","attribs":{"title":"P2rx7; Areg; Lilrb4b; Lag3; Itgb3; Lilrb4a; Sorl1; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr5; Epcam; Itga3; Cd36; Sell; Cd80; Ocln; Cd8a; Car4; Klrk1; Ptprk; Cd40; Ephb6; Cd74"},"children":["P2rx7; Areg; Lilrb4b; Lag3; Itgb3; Lilrb4a; Sorl1; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr..."]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Itgb3; Lilrb4a; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr5; Itga3; Cd36; Sell; Cd80; Cd8a; Car4; Klrk1; Cd40; Cd74"},"children":["P2rx7; Lag3; Itgb3; Lilrb4a; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr5; Itga3; Cd36; Sell; ..."]},{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Lag3; Lilrb4a; Sorl1; Itga2; Ccr4; Il1r2; Ptch1; Gpr15; Cxcr5; Ddr1; Cd36; Cd80; Vipr1; Ccr3; Klrk1; Reck; Cd40; Ephb6; Cd74"},"children":["P2rx7; Lilrb4b; Lag3; Lilrb4a; Sorl1; Itga2; Ccr4; Il1r2; Ptch1; Gpr15; Cxcr5; Ddr1; Cd36; Cd80; ..."]},{"name":"span","attribs":{"title":"Oxsr1; Itgb3; Ccl3; Ccl5; Itga2; Ccl4; S100a8; Ccr4; Ptch1; Egr2; Cxcr5; Bmp7; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Ear2; Ephb6; Cd74"},"children":["Oxsr1; Itgb3; Ccl3; Ccl5; Itga2; Ccl4; S100a8; Ccr4; Ptch1; Egr2; Cxcr5; Bmp7; Tnfsf14; Cxcl16; S..."]},{"name":"span","attribs":{"title":"Lgals7; Perp; Lilrb4b; Tespa1; Lag3; Twsg1; Itgb3; Dsp; Ccl5; Lilrb4a; Cd9; Itga2; Il10; S100a8; Serpine2; Cd24a; Epcam; Bmp7; Pkp4; Itga3; Tnfsf14; Sell; Cd80; S100a9; Ephb6; Cd74"},"children":["Lgals7; Perp; Lilrb4b; Tespa1; Lag3; Twsg1; Itgb3; Dsp; Ccl5; Lilrb4a; Cd9; Itga2; Il10; S100a8; ..."]},{"name":"span","attribs":{"title":"P2rx7; Lag3; Itgb3; Lilrb4a; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr5; Pkp4; Itga3; Cd36; Sell; Cd80; Cd8a; Car4; S100a6; Klrk1; Cd40; Cd74"},"children":["P2rx7; Lag3; Itgb3; Lilrb4a; Cd9; Itga2; Ccr4; Serpine2; Cd24a; Cd226; Cxcr5; Pkp4; Itga3; Cd36; ..."]},{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Lag3; Lilrb4a; Sorl1; Itga2; Ccr4; Il1r2; Ptch1; Gpr15; Cxcr5; Ddr1; Vipr1; Ccr3; Klrk1; Ephb6; Cd74"},"children":["P2rx7; Lilrb4b; Lag3; Lilrb4a; Sorl1; Itga2; Ccr4; Il1r2; Ptch1; Gpr15; Cxcr5; Ddr1; Vipr1; Ccr3;..."]},{"name":"span","attribs":{"title":"Oxsr1; Itgb3; Ccl3; Ccl5; Cd9; Itga2; Ccl4; S100a8; Cd24a; Gpr15; Cxcr5; Itga3; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Cd74"},"children":["Oxsr1; Itgb3; Ccl3; Ccl5; Cd9; Itga2; Ccl4; S100a8; Cd24a; Gpr15; Cxcr5; Itga3; Tnfsf14; Cxcl16; ..."]},{"name":"span","attribs":{"title":"Areg; Lilrb4b; Gcg; Ccl3; Ccl5; Lilrb4a; Il10; Ccl4; Insl6; Bmp7; Tnfsf14; Cxcl16"},"children":["Areg; Lilrb4b; Gcg; Ccl3; Ccl5; Lilrb4a; Il10; Ccl4; Insl6; Bmp7; Tnfsf14; Cxcl16"]},{"name":"span","attribs":{"title":"P2rx7; Cst7; Lilrb4b; Serpinf1; Ccl3; Ccl5; Lilrb4a; Itga2; Il10; Nkg7; Ccl4; S100a8; Ccr4; Il1r2; Dnase1l3; Cd24a; Ggt1; S100a9; Ccr3; Ifngr2; Cd40; Ephb6"},"children":["P2rx7; Cst7; Lilrb4b; Serpinf1; Ccl3; Ccl5; Lilrb4a; Itga2; Il10; Nkg7; Ccl4; S100a8; Ccr4; Il1r2..."]},{"name":"span","attribs":{"title":"Oxsr1; Ccl3; Ccl5; Ccl4; S100a8; Cxcr5; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Cd74"},"children":["Oxsr1; Ccl3; Ccl5; Ccl4; S100a8; Cxcr5; Tnfsf14; Cxcl16; Sell; S100a9; Ccr3; Cd74"]},{"name":"span","attribs":{"title":"Lilrb4b; Lilrb4a; Ccr4; Il1r2; Cxcr5; Ccr3; Klrk1; Cd74"},"children":["Lilrb4b; Lilrb4a; Ccr4; Il1r2; Cxcr5; Ccr3; Klrk1; Cd74"]},{"name":"span","attribs":{"title":"Trat1; Itgb3; Itga2; Nrn1; Ddr1; Itga3; Cd36; Cd8a; Vipr1; Cd40; Cd74"},"children":["Trat1; Itgb3; Itga2; Nrn1; Ddr1; Itga3; Cd36; Cd8a; Vipr1; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Itgb3; Dsp; Lilrb4a; Cd9; Itga2; Cd226; Epcam; Pkp4; Itga3; Sell"},"children":["Itgb3; Dsp; Lilrb4a; Cd9; Itga2; Cd226; Epcam; Pkp4; Itga3; Sell"]},{"name":"span","attribs":{"title":"Areg; Ccl3; Ccl5; Il10; Ccl4; Bmp7; Tnfsf14; Cxcl16"},"children":["Areg; Ccl3; Ccl5; Il10; Ccl4; Bmp7; Tnfsf14; Cxcl16"]},{"name":"span","attribs":{"title":"Areg; Gcg; Ccl3; Ccl5; Il10; Ccl4; Insl6; Bmp7; Tnfsf14; Cxcl16"},"children":["Areg; Gcg; Ccl3; Ccl5; Il10; Ccl4; Insl6; Bmp7; Tnfsf14; Cxcl16"]},{"name":"span","attribs":{"title":"Lilrb4b; Lilrb4a; Cd24a; Tnfsf14; Cd80; Ephb6"},"children":["Lilrb4b; Lilrb4a; Cd24a; Tnfsf14; Cd80; Ephb6"]},{"name":"span","attribs":{"title":"Lilrb4b; Tespa1; Lag3; Twsg1; Itgb3; Ccl5; Lilrb4a; Cd9; Itga2; Il10; Cass4; Serpine2; Cd24a; Epcam; Bmp7; Ddr1; Itga3; Cd36; Tnfsf14; Cd80; Ephb6; Cd74"},"children":["Lilrb4b; Tespa1; Lag3; Twsg1; Itgb3; Ccl5; Lilrb4a; Cd9; Itga2; Il10; Cass4; Serpine2; Cd24a; Epc..."]},{"name":"span","attribs":{"title":"Cst7; Serpinf1; Cd109; Serpine2; Tnfsf14; Arrb1; Reck"},"children":["Cst7; Serpinf1; Cd109; Serpine2; Tnfsf14; Arrb1; Reck"]},{"name":"span","attribs":{"title":"Areg; Gcg; Itgb3; Ccl3; Ccl5; Sorl1; Dgkg; Ccl4; Ccr4; Gpr15; Slc39a14; Cxcr5; Vipr1; Ccr3; Arrb1"},"children":["Areg; Gcg; Itgb3; Ccl3; Ccl5; Sorl1; Dgkg; Ccl4; Ccr4; Gpr15; Slc39a14; Cxcr5; Vipr1; Ccr3; Arrb1"]},{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Lag3; Lilrb4a; Eomes; Il10; Nkg7; Prf1; Dnase1l3; Cd24a; Cd226; Gzmm; Cd36; Cgas; Cd8a; Klrk1; Zpbp2; Cd40; Ephb6; Cd74"},"children":["P2rx7; Lilrb4b; Lag3; Lilrb4a; Eomes; Il10; Nkg7; Prf1; Dnase1l3; Cd24a; Cd226; Gzmm; Cd36; Cgas;..."]},{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Lag3; Lilrb4a; Nkg7; Prf1; Cd24a; Cd226; Gzmm; Cd8a; Klrk1; Cd40; Ephb6; Cd74"},"children":["P2rx7; Lilrb4b; Lag3; Lilrb4a; Nkg7; Prf1; Cd24a; Cd226; Gzmm; Cd8a; Klrk1; Cd40; Ephb6; Cd74"]},{"name":"span","attribs":{"title":"Lilrb4b; Itgb3; Lilrb4a; Itga3"},"children":["Lilrb4b; Itgb3; Lilrb4a; Itga3"]},{"name":"span","attribs":{"title":"Lilrb4b; Lag3; Twsg1; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Epcam; Cd80; Cd74"},"children":["Lilrb4b; Lag3; Twsg1; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Epcam; Cd80; Cd74"]},{"name":"span","attribs":{"title":"Oxsr1; Ccl3; Ccl5; Ccl4; Tnfsf14; Cxcl16"},"children":["Oxsr1; Ccl3; Ccl5; Ccl4; Tnfsf14; Cxcl16"]},{"name":"span","attribs":{"title":"Nkg7; Prf1; Cd24a; Cd226; Klrk1"},"children":["Nkg7; Prf1; Cd24a; Cd226; Klrk1"]},{"name":"span","attribs":{"title":"Areg; Itgb3; Ccl5; Cass4; Cd24a; Cd36; Cd80; Cd40; Cd74"},"children":["Areg; Itgb3; Ccl5; Cass4; Cd24a; Cd36; Cd80; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Oxsr1; Lag3; Ccl3; Ccl5; Itga2; Nkg7; S100a8; Ccr4; Cd24a; Cd226; Cd36; Tnfsf14; Cgas; Sell; S100a9; Klrk1; Cd40; Cd74"},"children":["Oxsr1; Lag3; Ccl3; Ccl5; Itga2; Nkg7; S100a8; Ccr4; Cd24a; Cd226; Cd36; Tnfsf14; Cgas; Sell; S100..."]},{"name":"span","attribs":{"title":"Twsg1; Itgb3; Il1r2; Cd109; Cd36; Ccr3; Cd74"},"children":["Twsg1; Itgb3; Il1r2; Cd109; Cd36; Ccr3; Cd74"]},{"name":"span","attribs":{"title":"Areg; Lilrb4b; Itgb3; Ccl5; Lilrb4a; Cass4; Cd24a; Cd36; Cd80; Cd40; Cd74"},"children":["Areg; Lilrb4b; Itgb3; Ccl5; Lilrb4a; Cass4; Cd24a; Cd36; Cd80; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Areg; Lilrb4b; Itgb3; Ccl5; Lilrb4a; Cass4; Cd24a; Ddr1; Cd36; Cd80; Cd40; Cd74"},"children":["Areg; Lilrb4b; Itgb3; Ccl5; Lilrb4a; Cass4; Cd24a; Ddr1; Cd36; Cd80; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Cst7; Lilrb4b; Tespa1; Tox; Lag3; Twsg1; Ccl5; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Cd226; Tnfsf14; Cgas; Cd80; Klrk1; Cd40; Ephb6; Cd74"},"children":["Cst7; Lilrb4b; Tespa1; Tox; Lag3; Twsg1; Ccl5; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Cd226; Tnfsf1..."]},{"name":"span","attribs":{"title":"Cst7; Lilrb4b; Lag3; Twsg1; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Cd80; Cd74"},"children":["Cst7; Lilrb4b; Lag3; Twsg1; Lilrb4a; Cd9; Il10; Serpine2; Cd24a; Cd80; Cd74"]},{"name":"span","attribs":{"title":"Ccl3; Ccl5; Ccl4; S100a8; Sell; S100a9; Ccr3; Cd74"},"children":["Ccl3; Ccl5; Ccl4; S100a8; Sell; S100a9; Ccr3; Cd74"]},{"name":"span","attribs":{"title":"Oxsr1; Itgb3; Ccl3; Ccl5; Ccl4; Gpr15; Tnfsf14; Cxcl16"},"children":["Oxsr1; Itgb3; Ccl3; Ccl5; Ccl4; Gpr15; Tnfsf14; Cxcl16"]}]}],"filterable":true,"dataKey":"7d1ddf16aad11651f51af404eae35082"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="CD49bvDP" class="section level3">
<h3 class="tabset">CD49bvDP</h3>
<em>All 2 ontologies considered enriched in the set of 33 DE genes mapped to EntrezIDs from the comparison CD49bvDP. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-322e5019c2fc6e0659b4" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-322e5019c2fc6e0659b4">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["CC","CC"],"ID":["GO:0005615","GO:0001533"],"Description":["extracellular space","cornified envelope"],"Count":[10,3],"% DE":[0.3125,0.09375],"% BG":[0.0453933050057001,0.0013472898745984],"p.adjust":[9.40781847267781e-05,0.000991448699816076],"geneID":["Fam20a; Igfbp7; Lgals7; Areg; S100a8; Cst7; Csf1; S100a9; Pla1a; Prrg4","Lgals7; Dsp; S100a9"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["9.41e-05","9.91e-04"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Fam20a; Igfbp7; Lgals7; Areg; S100a8; Cst7; Csf1; S100a9; Pla1a; Prrg4"},"children":["Fam20a; Igfbp7; Lgals7; Areg; S100a8; Cst7; Csf1; S100a9; Pla1a; Prrg4"]},{"name":"span","attribs":{"title":"Lgals7; Dsp; S100a9"},"children":["Lgals7; Dsp; S100a9"]}]}],"filterable":true,"dataKey":"add3e892f1154b9ccc286a7183010121"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="LAG3vDP" class="section level3">
<h3 class="tabset">LAG3vDP</h3>
<em>All 3 ontologies considered enriched in the set of 33 DE genes mapped to EntrezIDs from the comparison LAG3vDP. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-aae90d3e945975235f11" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-aae90d3e945975235f11">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["MF","MF","CC"],"ID":["GO:0004888","GO:0004930","GO:0009897"],"Description":["transmembrane signaling receptor activity","G protein-coupled receptor activity","external side of plasma membrane"],"Count":[7,4,5],"% DE":[0.21875,0.125,0.161290322580645],"% BG":[0.0207898035938153,0.00814876723777685,0.0218675510415587],"p.adjust":[0.000402381972870461,0.0151758840867833,0.0423014834980893],"geneID":["Cxcr5; Itga2; Klrk1; Gpr15; Vipr1; Gm5127; Cd79b","Cxcr5; Gpr15; Vipr1; Gm5127","Cxcr5; Itga2; Sell; Klrk1; Cd79b"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["4.02e-04","0.015","0.042"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Cxcr5; Itga2; Klrk1; Gpr15; Vipr1; Gm5127; Cd79b"},"children":["Cxcr5; Itga2; Klrk1; Gpr15; Vipr1; Gm5127; Cd79b"]},{"name":"span","attribs":{"title":"Cxcr5; Gpr15; Vipr1; Gm5127"},"children":["Cxcr5; Gpr15; Vipr1; Gm5127"]},{"name":"span","attribs":{"title":"Cxcr5; Itga2; Sell; Klrk1; Cd79b"},"children":["Cxcr5; Itga2; Sell; Klrk1; Cd79b"]}]}],"filterable":true,"dataKey":"5fdf9cb0b07b221d0232e1416dc70c52"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="enrichment-results-up-only" class="section level2 tabset">
<h2 class="tabset">Enrichment Results (Up Only)</h2>
<pre class="r"><code>ego_results &lt;- top_tables %&gt;% 
  lapply(
    function(x) {
      de &lt;- dplyr::filter(x, adj.P.Val &lt; 0.05, logFC &gt; 1)
      ids &lt;- unique(unlist(de$entrezid))
      enrichGO(
        ids,
        universe = universe,
        OrgDb = org.Mm.eg.db,
        ont = &quot;ALL&quot;,
        keyType = &quot;ENTREZID&quot;,
        pAdjustMethod = &quot;bonferroni&quot;,
        pvalueCutoff = 0.05,
        qvalueCutoff = 0.05,
        readable = TRUE
      )
    }
  )</code></pre>
<p>Enrichment testing was repeated using all three available Gene
Ontologies and the function <code>enrichGO()</code> from the package
<code>clusterProfiler</code>. Ontologies were only considered to be
enriched amongst the <strong>up-regulated genes</strong> if a
Bonferroni-adjusted p-value &lt; 0.05 was returned during enrichment
testing. Please note that an up-regulated gene does not always
correspond to an increase in activity of a pathway.</p>
<div id="dn-vs.-lag3-1" class="section level3">
<h3>DN Vs. Lag3<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$DNvLAG3) +
  ggtitle(expression(paste(&quot;DN Vs. Lag&quot;, 3^{textstyle(&quot;+&quot;)}, &quot;(Up Only)&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-dn-lag3-up-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-dn-lag3-up-1">
Past versions of dotplot-dn-lag3-up-1.png
</button>
</p>
<div id="fig-dotplot-dn-lag3-up-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-dn-lag3-up-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dn-vs.-dp-1" class="section level3">
<h3>DN Vs. DP</h3>
<p><em>No significant results were obtained for up-regulated genes in
this comparison.</em></p>
<pre class="r"><code>dotplot(ego_results$DNvDP) +
  ggtitle(&quot;DN Vs. DP (Up Only)&quot;) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
</div>
<div id="dn-vs.-cd49b-1" class="section level3">
<h3>DN Vs. Cd49b<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$DNvCD49b) +
  ggtitle(expression(paste(&quot;DN Vs. Cd49&quot;, b^{textstyle(&quot;+&quot;)}, &quot;(Up Only)&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-dn-cd49-up-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-dn-cd49-up-1">
Past versions of dotplot-dn-cd49-up-1.png
</button>
</p>
<div id="fig-dotplot-dn-cd49-up-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-dn-cd49-up-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="lag3-vs-cd49-1" class="section level3">
<h3>Lag3<sup>+</sup> Vs Cd49<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$LAG3vCD49b) +
  ggtitle(
    expression(
      paste(&quot;Lag&quot;, 3^{textstyle(&quot;+&quot;)}, &quot; Vs. CD49&quot;, b^{textstyle(&quot;+&quot;)}, &quot;(Up Only)&quot;)
    )
  ) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-lag3-cd49-up-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-lag3-cd49-up-1">
Past versions of dotplot-lag3-cd49-up-1.png
</button>
</p>
<div id="fig-dotplot-lag3-cd49-up-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-lag3-cd49-up-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="lag3-vs.-dp-1" class="section level3">
<h3>Lag3<sup>+</sup> Vs. DP</h3>
<p><em>No significant results were obtained for up-regulated genes in
this comparison.</em></p>
<pre class="r"><code>dotplot(ego_results$LAG3vDP) +
  ggtitle(expression(paste(&quot;Lag&quot;, 3^{textstyle(&quot;+&quot;)}, &quot; Vs. DP&quot;, &quot;(Up Only)&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
</div>
<div id="cd49b-vs.-dp-1" class="section level3">
<h3>Cd49b<sup>+</sup> Vs. DP</h3>
<pre class="r"><code>dotplot(ego_results$CD49bvDP) +
  ggtitle(expression(paste(&quot;Cd49&quot;, b^{textstyle(&quot;+&quot;)}, &quot; Vs. DP (Up Only)&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-cd49-dp-up-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-cd49-dp-up-1">
Past versions of dotplot-cd49-dp-up-1.png
</button>
</p>
<div id="fig-dotplot-cd49-dp-up-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-cd49-dp-up-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="enrichment-tables-up-only" class="section level2 tabset">
<h2 class="tabset">Enrichment Tables (Up Only)</h2>
<pre class="r"><code>ego_results &lt;- ego_results[vapply(ego_results, nrow, integer(1)) &gt; 0]
htmltools::tagList(
  names(ego_results) %&gt;% 
    lapply(
      function(x) {
        df &lt;- ego_results[[x]] %&gt;% 
          as_tibble() %&gt;% 
          dplyr::filter(p.adjust &lt; 0.05) %&gt;% 
          mutate(
            geneID = str_replace_all(geneID, &quot;\\/&quot;, &quot;; &quot;),
            `% DE` = vapply(GeneRatio, function(x) eval(parse(text = x)), numeric(1)),
            `% BG` = vapply(BgRatio, function(x) eval(parse(text = x)), numeric(1))
          ) %&gt;% 
          dplyr::select(ONTOLOGY, ID, Description, Count, starts_with(&quot;%&quot;), p.adjust, geneID) %&gt;% 
          arrange(p.adjust) %&gt;% 
          distinct(ONTOLOGY, geneID, .keep_all = TRUE) 
        
        if (nrow(df)) {
          cp &lt;- glue(
            &quot;All {nrow(df)} ontologies considered enriched in the set of &quot;,
            length(unique(unlist(dplyr::filter(top_tables[[x]], DE, logFC &gt; 1)$entrezid))),
            &quot; up-regulated genes mapped to EntrezIDs from the comparison {x}. Enrichment &quot;,
            &quot;testing was performed using Fisher&#39;s Exact Test compared to the set of &quot;,
            comma(length(unique(universe))), 
            &quot; unique EntrezIDs mapped to genes considered as detectable. &quot;,
            &quot;The percentage of nonDE genes mapped to the ontology are also given. &quot;,
            &quot;P-values are Bonferroni adjusted and all pathways are considered enriched &quot;,
            &quot;using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your &quot;, 
            &quot;mouse over the final column. &quot;,
            &quot;Where the identical genes map to multiple terms within each ontology, only &quot;, 
            &quot;the term with the lowest p-value (i.e. strongest enrichment) is shown. &quot;,
            &quot;As a result, some of the pathways seen in the above dotplots may not be &quot;,
            &quot;presented in these tables.&quot;
          )
          htmltools::div(
            htmltools::div(
              id = glue(&quot;{x}-up&quot;), 
              class=&quot;section level3&quot;,
              htmltools::h3(class = &quot;tabset&quot;, x),
              htmltools::tags$em(cp),
              df %&gt;% 
                reactable(
                  filterable = TRUE,
                  columns = list(
                    ONTOLOGY = colDef(name = &quot;Ontology&quot;, maxWidth = 100),
                    ID = colDef(name = &quot;GO ID&quot;, maxWidth = 125),
                    Description = colDef(maxWidth = 250),
                    Count = colDef(name = &quot;N&lt;sub&gt;DE&lt;/sub&gt;&quot;, html = TRUE, maxWidth = 80),
                    `% DE` = colDef(format = colFormat(percent = TRUE, digits = 1), maxWidth = 85),
                    `% BG` = colDef(format = colFormat(percent = TRUE, digits = 1), maxWidth = 85),
                    p.adjust = colDef(
                      name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE, maxWidth = 100,
                      cell = function(value) {
                        fm &lt;- ifelse(value &lt; 0.001, &quot;%.2e&quot;, &quot;%.3f&quot;)
                        sprintf(fm, value)
                      }
                    ),
                    geneID = colDef(
                      name = &quot;DE Genes&quot;,
                      cell = function(value) with_tooltip(value, width = 100)
                    )
                  )
                )
            )
          )
        } else {
          cat(glue(&quot;No enrichment was found for {x}\n\n&quot;))
        }
      }
    )
)</code></pre>
<div>
<div id="DNvLAG3-up" class="section level3">
<h3 class="tabset">DNvLAG3</h3>
<em>All 55 ontologies considered enriched in the set of 45 up-regulated genes mapped to EntrezIDs from the comparison DNvLAG3. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-3ad505f8ec5d90b839e0" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-3ad505f8ec5d90b839e0">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["CC","CC","BP","BP","CC","MF","BP","MF","CC","CC","MF","BP","CC","MF","BP","CC","MF","BP","BP","BP","BP","BP","BP","BP","BP","BP","MF","CC","BP","BP","BP","BP","BP","BP","MF","BP","BP","BP","BP","BP","BP","BP","BP","BP","BP","BP","BP","BP","MF","BP","BP","BP","BP","BP","BP"],"ID":["GO:0009897","GO:0098552","GO:0034341","GO:0006954","GO:0009986","GO:0023026","GO:0002250","GO:0005126","GO:0005615","GO:0098797","GO:0005125","GO:0022407","GO:0042611","GO:0008009","GO:0071347","GO:0043235","GO:0038023","GO:0050778","GO:0051249","GO:1903131","GO:0070098","GO:0019886","GO:0030595","GO:0032496","GO:0002768","GO:0030098","GO:0003823","GO:0098802","GO:0050863","GO:0002443","GO:0022409","GO:0002449","GO:0030593","GO:0043408","GO:0042605","GO:0002253","GO:0014048","GO:0033993","GO:0051251","GO:0006935","GO:0043405","GO:0042110","GO:0050870","GO:0002429","GO:0019221","GO:0002396","GO:0034612","GO:0006959","GO:0004888","GO:0019730","GO:0002440","GO:0071222","GO:0009617","GO:0097529","GO:0045580"],"Description":["external side of plasma membrane","side of membrane","response to type II interferon","inflammatory response","cell surface","MHC class II protein complex binding","adaptive immune response","cytokine receptor binding","extracellular space","plasma membrane protein complex","cytokine activity","regulation of cell-cell adhesion","MHC protein complex","chemokine activity","cellular response to interleukin-1","receptor complex","signaling receptor activity","positive regulation of immune response","regulation of lymphocyte activation","mononuclear cell differentiation","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class II","leukocyte chemotaxis","response to lipopolysaccharide","immune response-regulating cell surface receptor signaling pathway","lymphocyte differentiation","antigen binding","plasma membrane signaling receptor complex","regulation of T cell activation","leukocyte mediated immunity","positive regulation of cell-cell adhesion","lymphocyte mediated immunity","neutrophil chemotaxis","regulation of MAPK cascade","peptide antigen binding","activation of immune response","regulation of glutamate secretion","response to lipid","positive regulation of lymphocyte activation","chemotaxis","regulation of MAP kinase activity","T cell activation","positive regulation of T cell activation","immune response-activating cell surface receptor signaling pathway","cytokine-mediated signaling pathway","MHC protein complex assembly","response to tumor necrosis factor","humoral immune response","transmembrane signaling receptor activity","antimicrobial humoral response","production of molecular mediator of immune response","cellular response to lipopolysaccharide","response to bacterium","myeloid leukocyte migration","regulation of T cell differentiation"],"Count":[11,12,9,14,12,4,12,8,12,9,6,11,4,4,6,7,9,12,11,11,5,4,7,8,8,10,4,5,9,9,8,8,5,10,3,9,3,10,8,8,6,10,7,7,8,3,6,5,6,4,7,6,9,6,6],"% DE":[0.261904761904762,0.285714285714286,0.209302325581395,0.325581395348837,0.285714285714286,0.0975609756097561,0.27906976744186,0.195121951219512,0.285714285714286,0.214285714285714,0.146341463414634,0.255813953488372,0.0952380952380952,0.0975609756097561,0.13953488372093,0.166666666666667,0.219512195121951,0.27906976744186,0.255813953488372,0.255813953488372,0.116279069767442,0.0930232558139535,0.162790697674419,0.186046511627907,0.186046511627907,0.232558139534884,0.0975609756097561,0.119047619047619,0.209302325581395,0.209302325581395,0.186046511627907,0.186046511627907,0.116279069767442,0.232558139534884,0.0731707317073171,0.209302325581395,0.0697674418604651,0.232558139534884,0.186046511627907,0.186046511627907,0.13953488372093,0.232558139534884,0.162790697674419,0.162790697674419,0.186046511627907,0.0697674418604651,0.13953488372093,0.116279069767442,0.146341463414634,0.0930232558139535,0.162790697674419,0.13953488372093,0.209302325581395,0.13953488372093,0.13953488372093],"% BG":[0.0218675510415587,0.0353404497875427,0.0114004811212216,0.0440330509360946,0.0403150585552907,0.00104471374843293,0.0338876686539065,0.0144170497283744,0.0453933050057001,0.0220748264068815,0.00720852486418721,0.0333647108043092,0.00217639133588973,0.00198495612202257,0.00585712791549001,0.0150274639859053,0.0300877559548684,0.0474845727434369,0.0385942893002824,0.0392218387197992,0.00386988808702019,0.0017780566886309,0.013283129379772,0.0193494404351009,0.0196632151448593,0.0351427674929401,0.00376096949435855,0.0086019276608975,0.0283443154481749,0.0290764564376111,0.0226963706725238,0.0231147369522017,0.0061709026252484,0.0411044869783495,0.00198495612202257,0.0328417529547119,0.00104591569919465,0.0430917268068194,0.0255203430603493,0.0257295262001883,0.0122372136805773,0.0442422340759335,0.0189310741554231,0.0192448488651815,0.0268800334693024,0.00125509883903357,0.0130739462399331,0.00784436774395984,0.0207898035938153,0.00397447965693965,0.0208137224139734,0.0142244535090472,0.0384896977303629,0.0144336366488861,0.014642819788725],"p.adjust":[9.76077963411674e-08,1.30493316055535e-06,1.35101517191117e-06,2.87256558352942e-06,5.6344741661014e-06,1.00922173758059e-05,1.58727054091851e-05,1.64661872023034e-05,2.06229905304377e-05,2.44145707759786e-05,7.00613536806588e-05,0.00015105472203481,0.000172381339920974,0.00018115300829362,0.000224851308409429,0.000254884791320401,0.000452909277821653,0.000645821007620829,0.000655580251999638,0.000770023010202034,0.000806151411383656,0.00114957103138178,0.00190218346052764,0.00195017551139527,0.00220166848154728,0.00243547227761998,0.00261192794570199,0.0027080889193528,0.0035588913235693,0.00439103171438597,0.00643492374739505,0.0073674083905881,0.00860701885387382,0.00990708798485996,0.011390385951379,0.011859632381937,0.0141367620370363,0.0150200864389858,0.015263192599355,0.0162014657910502,0.0176778633507886,0.0189196323016694,0.0199787719791713,0.0222358915034255,0.0222804644123696,0.025755111361748,0.025815132424911,0.0281384802919318,0.0310205603623087,0.0332901584866544,0.0369373054758364,0.041693342593416,0.0421718620331884,0.0452821003635941,0.0491154697856485],"geneID":["P2rx7; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4","P2rx7; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd14; Cd79a; Ccr4","Acod1; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cxcl16; Ccl8; Ccl7","P2rx7; Acod1; Cd40; Lyn; Cxcl2; Il1b; Il1rn; Cd14; Chil5; Pld4; Ccl8; Ifngr2; Ccr4; Ccl7","P2rx7; Plaur; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4","H2-Ab1; Cd74; H2-Aa; H2-Eb1","P2rx7; Trat1; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b; Cd79a","Lyn; Cxcl2; Il1b; Il1rn; Cxcl16; Sh2b3; Ccl8; Ccl7","Cd74; Cd40; Cxcl2; Il1b; Il1rn; Slpi; Cd14; Cxcl16; Chil5; Ccl8; Klk1b11; Ccl7","Trat1; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd79a","Cxcl2; Il1b; Il1rn; Cxcl16; Ccl8; Ccl7","Tespa1; Plaur; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; Sh2b3","H2-Ab1; Cd74; H2-Aa; H2-Eb1","Cxcl2; Cxcl16; Ccl8; Ccl7","Acod1; Cd40; Il1b; Il1rn; Ccl8; Ccl7","Trat1; Cd79b; Cd74; Cd40; Lyn; Cd14; Cd79a","P2rx7; Plaur; Cd79b; Cd74; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4","Tespa1; P2rx7; Acod1; Trat1; Cd79b; H2-Ab1; Cd74; Cd40; Lyn; Il1b; Cd14; Cd79a","Tespa1; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3","Tespa1; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Lyn; Il1b; Sh2b3; Cd79a","Lyn; Cxcl2; Sh2b3; Ccl8; Ccl7","H2-Ab1; Cd74; H2-Aa; H2-Eb1","Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccl7","P2rx7; Acod1; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16","Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Cd14; Cd79a","Tespa1; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Il1b; Sh2b3; Cd79a","H2-Ab1; H2-Aa; Cd40; H2-Eb1","Trat1; Cd79b; Cd40; Lyn; Cd79a","Tespa1; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3","P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b","Tespa1; Plaur; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b","P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Gzmm; Il1b","Cd74; Cxcl2; Il1b; Ccl8; Ccl7","P2rx7; Atf3; Cd74; Cd40; Lyn; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7","H2-Ab1; H2-Aa; H2-Eb1","Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Il1b; Cd14; Cd79a","P2rx7; Il1b; Il1rn","Id3; P2rx7; Acod1; Lyn; Cxcl2; Il1b; Il1rn; Slpi; Cd14; Cxcl16","Tespa1; Cd83; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b","Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccr4; Ccl7","P2rx7; Cd40; Lyn; Il1b; Il1rn; Sh2b3","Tespa1; P2rx7; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3","Tespa1; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b","Tespa1; Acod1; Trat1; Cd79b; Lyn; Cd14; Cd79a","Cd74; Lyn; Cxcl2; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7","H2-Ab1; H2-Aa; H2-Eb1","Acod1; Cd40; Cd14; Cxcl16; Ccl8; Ccl7","Acod1; Cxcl2; Il1b; Slpi; Ccl8","P2rx7; Cd79b; Cd74; H2-Eb1; Cd79a; Ccr4","Acod1; Cxcl2; Slpi; Ccl8","P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b","Acod1; Lyn; Cxcl2; Il1b; Cd14; Cxcl16","P2rx7; Acod1; Cd79b; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16","Cd74; Lyn; Cxcl2; Il1b; Ccl8; Ccl7","Tespa1; Cd83; Tcf7; Cd74; H2-Aa; Sh2b3"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["9.76e-08","1.30e-06","1.35e-06","2.87e-06","5.63e-06","1.01e-05","1.59e-05","1.65e-05","2.06e-05","2.44e-05","7.01e-05","1.51e-04","1.72e-04","1.81e-04","2.25e-04","2.55e-04","4.53e-04","6.46e-04","6.56e-04","7.70e-04","8.06e-04","0.001","0.002","0.002","0.002","0.002","0.003","0.003","0.004","0.004","0.006","0.007","0.009","0.010","0.011","0.012","0.014","0.015","0.015","0.016","0.018","0.019","0.020","0.022","0.022","0.026","0.026","0.028","0.031","0.033","0.037","0.042","0.042","0.045","0.049"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"P2rx7; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4"},"children":["P2rx7; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"P2rx7; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd14; Cd79a; Ccr4"},"children":["P2rx7; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd14; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"Acod1; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cxcl16; Ccl8; Ccl7"},"children":["Acod1; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"P2rx7; Acod1; Cd40; Lyn; Cxcl2; Il1b; Il1rn; Cd14; Chil5; Pld4; Ccl8; Ifngr2; Ccr4; Ccl7"},"children":["P2rx7; Acod1; Cd40; Lyn; Cxcl2; Il1b; Il1rn; Cd14; Chil5; Pld4; Ccl8; Ifngr2; Ccr4; Ccl7"]},{"name":"span","attribs":{"title":"P2rx7; Plaur; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4"},"children":["P2rx7; Plaur; Cd83; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"H2-Ab1; Cd74; H2-Aa; H2-Eb1"},"children":["H2-Ab1; Cd74; H2-Aa; H2-Eb1"]},{"name":"span","attribs":{"title":"P2rx7; Trat1; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b; Cd79a"},"children":["P2rx7; Trat1; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b; Cd79a"]},{"name":"span","attribs":{"title":"Lyn; Cxcl2; Il1b; Il1rn; Cxcl16; Sh2b3; Ccl8; Ccl7"},"children":["Lyn; Cxcl2; Il1b; Il1rn; Cxcl16; Sh2b3; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Cd74; Cd40; Cxcl2; Il1b; Il1rn; Slpi; Cd14; Cxcl16; Chil5; Ccl8; Klk1b11; Ccl7"},"children":["Cd74; Cd40; Cxcl2; Il1b; Il1rn; Slpi; Cd14; Cxcl16; Chil5; Ccl8; Klk1b11; Ccl7"]},{"name":"span","attribs":{"title":"Trat1; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd79a"},"children":["Trat1; Cd79b; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Cd79a"]},{"name":"span","attribs":{"title":"Cxcl2; Il1b; Il1rn; Cxcl16; Ccl8; Ccl7"},"children":["Cxcl2; Il1b; Il1rn; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Tespa1; Plaur; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; Sh2b3"},"children":["Tespa1; Plaur; Cd83; H2-Ab1; Cd74; H2-Aa; Lyn; H2-Eb1; Il1b; Il1rn; Sh2b3"]},{"name":"span","attribs":{"title":"H2-Ab1; Cd74; H2-Aa; H2-Eb1"},"children":["H2-Ab1; Cd74; H2-Aa; H2-Eb1"]},{"name":"span","attribs":{"title":"Cxcl2; Cxcl16; Ccl8; Ccl7"},"children":["Cxcl2; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Acod1; Cd40; Il1b; Il1rn; Ccl8; Ccl7"},"children":["Acod1; Cd40; Il1b; Il1rn; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Trat1; Cd79b; Cd74; Cd40; Lyn; Cd14; Cd79a"},"children":["Trat1; Cd79b; Cd74; Cd40; Lyn; Cd14; Cd79a"]},{"name":"span","attribs":{"title":"P2rx7; Plaur; Cd79b; Cd74; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4"},"children":["P2rx7; Plaur; Cd79b; Cd74; Cd40; H2-Eb1; Cd14; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"Tespa1; P2rx7; Acod1; Trat1; Cd79b; H2-Ab1; Cd74; Cd40; Lyn; Il1b; Cd14; Cd79a"},"children":["Tespa1; P2rx7; Acod1; Trat1; Cd79b; H2-Ab1; Cd74; Cd40; Lyn; Il1b; Cd14; Cd79a"]},{"name":"span","attribs":{"title":"Tespa1; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3"},"children":["Tespa1; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Il1b; Sh2b3"]},{"name":"span","attribs":{"title":"Tespa1; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Lyn; Il1b; Sh2b3; Cd79a"},"children":["Tespa1; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Lyn; Il1b; Sh2b3; Cd79a"]},{"name":"span","attribs":{"title":"Lyn; Cxcl2; Sh2b3; Ccl8; Ccl7"},"children":["Lyn; Cxcl2; Sh2b3; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"H2-Ab1; Cd74; H2-Aa; H2-Eb1"},"children":["H2-Ab1; Cd74; H2-Aa; H2-Eb1"]},{"name":"span","attribs":{"title":"Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccl7"},"children":["Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"P2rx7; Acod1; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16"},"children":["P2rx7; Acod1; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16"]},{"name":"span","attribs":{"title":"Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Cd14; Cd79a"},"children":["Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Cd14; Cd79a"]},{"name":"span","attribs":{"title":"Tespa1; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Il1b; Sh2b3; Cd79a"},"children":["Tespa1; Cd83; Cd79b; Tcf7; H2-Ab1; Cd74; H2-Aa; Il1b; Sh2b3; Cd79a"]},{"name":"span","attribs":{"title":"H2-Ab1; H2-Aa; Cd40; H2-Eb1"},"children":["H2-Ab1; H2-Aa; Cd40; H2-Eb1"]},{"name":"span","attribs":{"title":"Trat1; Cd79b; Cd40; Lyn; Cd79a"},"children":["Trat1; Cd79b; Cd40; Lyn; Cd79a"]},{"name":"span","attribs":{"title":"Tespa1; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3"},"children":["Tespa1; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3"]},{"name":"span","attribs":{"title":"P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b"},"children":["P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; Lyn; H2-Eb1; Gzmm; Il1b"]},{"name":"span","attribs":{"title":"Tespa1; Plaur; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"},"children":["Tespa1; Plaur; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"]},{"name":"span","attribs":{"title":"P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Gzmm; Il1b"},"children":["P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Gzmm; Il1b"]},{"name":"span","attribs":{"title":"Cd74; Cxcl2; Il1b; Ccl8; Ccl7"},"children":["Cd74; Cxcl2; Il1b; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"P2rx7; Atf3; Cd74; Cd40; Lyn; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7"},"children":["P2rx7; Atf3; Cd74; Cd40; Lyn; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"H2-Ab1; H2-Aa; H2-Eb1"},"children":["H2-Ab1; H2-Aa; H2-Eb1"]},{"name":"span","attribs":{"title":"Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Il1b; Cd14; Cd79a"},"children":["Tespa1; Acod1; Trat1; Cd79b; Cd40; Lyn; Il1b; Cd14; Cd79a"]},{"name":"span","attribs":{"title":"P2rx7; Il1b; Il1rn"},"children":["P2rx7; Il1b; Il1rn"]},{"name":"span","attribs":{"title":"Id3; P2rx7; Acod1; Lyn; Cxcl2; Il1b; Il1rn; Slpi; Cd14; Cxcl16"},"children":["Id3; P2rx7; Acod1; Lyn; Cxcl2; Il1b; Il1rn; Slpi; Cd14; Cxcl16"]},{"name":"span","attribs":{"title":"Tespa1; Cd83; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b"},"children":["Tespa1; Cd83; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b"]},{"name":"span","attribs":{"title":"Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccr4; Ccl7"},"children":["Cd74; Lyn; Cxcl2; Il1b; Cxcl16; Ccl8; Ccr4; Ccl7"]},{"name":"span","attribs":{"title":"P2rx7; Cd40; Lyn; Il1b; Il1rn; Sh2b3"},"children":["P2rx7; Cd40; Lyn; Il1b; Il1rn; Sh2b3"]},{"name":"span","attribs":{"title":"Tespa1; P2rx7; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3"},"children":["Tespa1; P2rx7; Cd83; Tcf7; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b; Sh2b3"]},{"name":"span","attribs":{"title":"Tespa1; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"},"children":["Tespa1; Cd83; H2-Ab1; Cd74; H2-Aa; H2-Eb1; Il1b"]},{"name":"span","attribs":{"title":"Tespa1; Acod1; Trat1; Cd79b; Lyn; Cd14; Cd79a"},"children":["Tespa1; Acod1; Trat1; Cd79b; Lyn; Cd14; Cd79a"]},{"name":"span","attribs":{"title":"Cd74; Lyn; Cxcl2; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7"},"children":["Cd74; Lyn; Cxcl2; Il1b; Il1rn; Sh2b3; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"H2-Ab1; H2-Aa; H2-Eb1"},"children":["H2-Ab1; H2-Aa; H2-Eb1"]},{"name":"span","attribs":{"title":"Acod1; Cd40; Cd14; Cxcl16; Ccl8; Ccl7"},"children":["Acod1; Cd40; Cd14; Cxcl16; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Acod1; Cxcl2; Il1b; Slpi; Ccl8"},"children":["Acod1; Cxcl2; Il1b; Slpi; Ccl8"]},{"name":"span","attribs":{"title":"P2rx7; Cd79b; Cd74; H2-Eb1; Cd79a; Ccr4"},"children":["P2rx7; Cd79b; Cd74; H2-Eb1; Cd79a; Ccr4"]},{"name":"span","attribs":{"title":"Acod1; Cxcl2; Slpi; Ccl8"},"children":["Acod1; Cxcl2; Slpi; Ccl8"]},{"name":"span","attribs":{"title":"P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b"},"children":["P2rx7; H2-Ab1; Cd74; H2-Aa; Cd40; H2-Eb1; Il1b"]},{"name":"span","attribs":{"title":"Acod1; Lyn; Cxcl2; Il1b; Cd14; Cxcl16"},"children":["Acod1; Lyn; Cxcl2; Il1b; Cd14; Cxcl16"]},{"name":"span","attribs":{"title":"P2rx7; Acod1; Cd79b; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16"},"children":["P2rx7; Acod1; Cd79b; Lyn; Cxcl2; Il1b; Slpi; Cd14; Cxcl16"]},{"name":"span","attribs":{"title":"Cd74; Lyn; Cxcl2; Il1b; Ccl8; Ccl7"},"children":["Cd74; Lyn; Cxcl2; Il1b; Ccl8; Ccl7"]},{"name":"span","attribs":{"title":"Tespa1; Cd83; Tcf7; Cd74; H2-Aa; Sh2b3"},"children":["Tespa1; Cd83; Tcf7; Cd74; H2-Aa; Sh2b3"]}]}],"filterable":true,"dataKey":"dbb5f25d079234b5210b185bd4cf04a8"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="DNvCD49b-up" class="section level3">
<h3 class="tabset">DNvCD49b</h3>
<em>All 11 ontologies considered enriched in the set of 14 up-regulated genes mapped to EntrezIDs from the comparison DNvCD49b. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-b62ee322197559a7ac80" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-b62ee322197559a7ac80">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["BP","CC","MF","BP","BP","BP","MF","BP","BP","BP","BP"],"ID":["GO:0002523","GO:0009897","GO:0005504","GO:1903034","GO:0030193","GO:0050900","GO:0036041","GO:1903036","GO:0048708","GO:0030194","GO:0034111"],"Description":["leukocyte migration involved in inflammatory response","external side of plasma membrane","fatty acid binding","regulation of response to wounding","regulation of blood coagulation","leukocyte migration","long-chain fatty acid binding","positive regulation of response to wounding","astrocyte differentiation","positive regulation of blood coagulation","negative regulation of homotypic cell-cell adhesion"],"Count":[3,5,3,4,3,5,2,3,3,2,2],"% DE":[0.25,0.384615384615385,0.230769230769231,0.333333333333333,0.25,0.416666666666667,0.153846153846154,0.25,0.25,0.166666666666667,0.166666666666667],"% BG":[0.00125509883903357,0.0218675510415587,0.00250731299623903,0.0079489593138793,0.00251019767806715,0.0230101453822822,0.00104471374843293,0.00376529651710072,0.00491580378621483,0.00104591569919465,0.00104591569919465],"p.adjust":[0.00024241376380633,0.000303296526472827,0.000319533531439463,0.00127643581606276,0.00221136988843874,0.00314149488908803,0.00624493266628178,0.00773506165253324,0.0174302216738646,0.0474348387436324,0.0474348387436324],"geneID":["S100a8; Cd24a; S100a9","Cd24a; Cxcr5; Cd9; Cd36; Cd8a","S100a8; Id3; Cd36","Cd24a; S100a9; Cd9; Cd36","S100a9; Cd9; Cd36","S100a8; Cd24a; Cxcr5; S100a9; Cd9","S100a8; Cd36","Cd24a; S100a9; Cd36","S100a8; Eomes; S100a9","S100a9; Cd36","Cd24a; Cd9"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["2.42e-04","3.03e-04","3.20e-04","0.001","0.002","0.003","0.006","0.008","0.017","0.047","0.047"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"S100a8; Cd24a; S100a9"},"children":["S100a8; Cd24a; S100a9"]},{"name":"span","attribs":{"title":"Cd24a; Cxcr5; Cd9; Cd36; Cd8a"},"children":["Cd24a; Cxcr5; Cd9; Cd36; Cd8a"]},{"name":"span","attribs":{"title":"S100a8; Id3; Cd36"},"children":["S100a8; Id3; Cd36"]},{"name":"span","attribs":{"title":"Cd24a; S100a9; Cd9; Cd36"},"children":["Cd24a; S100a9; Cd9; Cd36"]},{"name":"span","attribs":{"title":"S100a9; Cd9; Cd36"},"children":["S100a9; Cd9; Cd36"]},{"name":"span","attribs":{"title":"S100a8; Cd24a; Cxcr5; S100a9; Cd9"},"children":["S100a8; Cd24a; Cxcr5; S100a9; Cd9"]},{"name":"span","attribs":{"title":"S100a8; Cd36"},"children":["S100a8; Cd36"]},{"name":"span","attribs":{"title":"Cd24a; S100a9; Cd36"},"children":["Cd24a; S100a9; Cd36"]},{"name":"span","attribs":{"title":"S100a8; Eomes; S100a9"},"children":["S100a8; Eomes; S100a9"]},{"name":"span","attribs":{"title":"S100a9; Cd36"},"children":["S100a9; Cd36"]},{"name":"span","attribs":{"title":"Cd24a; Cd9"},"children":["Cd24a; Cd9"]}]}],"filterable":true,"dataKey":"f433831f51e231c30fa8147de6372a20"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="LAG3vCD49b-up" class="section level3">
<h3 class="tabset">LAG3vCD49b</h3>
<em>All 13 ontologies considered enriched in the set of 99 up-regulated genes mapped to EntrezIDs from the comparison LAG3vCD49b. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-446a7f2c4327c71dcaa6" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-446a7f2c4327c71dcaa6">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["CC","CC","BP","CC","MF","BP","BP","BP","CC","CC","MF","MF","BP"],"ID":["GO:0005615","GO:0009986","GO:0098609","GO:0009897","GO:0005509","GO:0098742","GO:1903034","GO:0050900","GO:0098590","GO:0098552","GO:0048018","GO:0050431","GO:0030593"],"Description":["extracellular space","cell surface","cell-cell adhesion","external side of plasma membrane","calcium ion binding","cell-cell adhesion via plasma-membrane adhesion molecules","regulation of response to wounding","leukocyte migration","plasma membrane region","side of membrane","receptor ligand activity","transforming growth factor beta binding","neutrophil chemotaxis"],"Count":[25,16,17,11,10,6,7,11,15,12,7,3,6],"% DE":[0.255102040816327,0.163265306122449,0.180851063829787,0.112244897959184,0.104166666666667,0.0638297872340425,0.074468085106383,0.117021276595745,0.153061224489796,0.122448979591837,0.0729166666666667,0.03125,0.0638297872340425],"% BG":[0.0453933050057001,0.0403150585552907,0.0442422340759335,0.0218675510415587,0.0203719180944421,0.00460202907645644,0.0079489593138793,0.0230101453822822,0.051611565965385,0.0353404497875427,0.0120142081069787,0.00114918512327622,0.0061709026252484],"p.adjust":[1.59563689060552e-10,0.000306410946923287,0.00128403241417308,0.00166871256180817,0.00715577391889174,0.00820219967552319,0.0182108551560144,0.0196678608746688,0.0257740440611686,0.0308518168546937,0.0441052233125424,0.0451970075676386,0.0465061857931611],"geneID":["Lgals7; Igfbp7; Areg; Cst7; Angptl2; Serpinf1; Fam20a; Twsg1; Gcg; Ccl3; Ccl5; Cd9; Nrn1; Il10; Ccl4; S100a8; Prf1; Cd109; Serpine2; Pla1a; Sell; S100a9; Car4; Ear2; Prrg4","Areg; Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Epcam; Itga3; Cd36; Sell; Cd80; Ocln; Cd8a; Car4; Ptprk; Ephb6","Lgals7; Perp; Lag3; Twsg1; Dsp; Ccl5; Cd9; Il10; S100a8; Serpine2; Cd24a; Epcam; Itga3; Sell; Cd80; S100a9; Ephb6","Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Itga3; Cd36; Sell; Cd80; Cd8a; Car4","Dgkg; S100a8; S100a1; Prf1; Cetn4; Sell; S100a9; S100a6; Prrg4; Ncald","Lgals7; Il10; Cd24a; Epcam; Itga3; Sell","Cd9; Il10; Cd109; Serpine2; Cd24a; Cd36; S100a9","Oxsr1; Ccl3; Ccl5; Cd9; Ccl4; S100a8; Cd24a; Cxcr5; Itga3; Sell; S100a9","Dsp; Cd9; Dlg2; Ptch1; Cd24a; Ptpn5; Slc39a14; Epcam; Ddr1; Itga3; Cd36; Ocln; Cd8a; Car4; Arrb1","Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Itga3; Cd36; Sell; Cd80; Cd8a; Car4; S100a6","Areg; Gcg; Ccl3; Ccl5; Il10; Ccl4; Insl6","Twsg1; Cd109; Cd36","Ccl3; Ccl5; Ccl4; S100a8; Sell; S100a9"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["1.60e-10","3.06e-04","0.001","0.002","0.007","0.008","0.018","0.020","0.026","0.031","0.044","0.045","0.047"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Lgals7; Igfbp7; Areg; Cst7; Angptl2; Serpinf1; Fam20a; Twsg1; Gcg; Ccl3; Ccl5; Cd9; Nrn1; Il10; Ccl4; S100a8; Prf1; Cd109; Serpine2; Pla1a; Sell; S100a9; Car4; Ear2; Prrg4"},"children":["Lgals7; Igfbp7; Areg; Cst7; Angptl2; Serpinf1; Fam20a; Twsg1; Gcg; Ccl3; Ccl5; Cd9; Nrn1; Il10; C..."]},{"name":"span","attribs":{"title":"Areg; Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Epcam; Itga3; Cd36; Sell; Cd80; Ocln; Cd8a; Car4; Ptprk; Ephb6"},"children":["Areg; Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Epcam; Itga3; Cd36; Sell; Cd80; Ocln; Cd8a; Car4; Ptprk;..."]},{"name":"span","attribs":{"title":"Lgals7; Perp; Lag3; Twsg1; Dsp; Ccl5; Cd9; Il10; S100a8; Serpine2; Cd24a; Epcam; Itga3; Sell; Cd80; S100a9; Ephb6"},"children":["Lgals7; Perp; Lag3; Twsg1; Dsp; Ccl5; Cd9; Il10; S100a8; Serpine2; Cd24a; Epcam; Itga3; Sell; Cd8..."]},{"name":"span","attribs":{"title":"Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Itga3; Cd36; Sell; Cd80; Cd8a; Car4"},"children":["Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Itga3; Cd36; Sell; Cd80; Cd8a; Car4"]},{"name":"span","attribs":{"title":"Dgkg; S100a8; S100a1; Prf1; Cetn4; Sell; S100a9; S100a6; Prrg4; Ncald"},"children":["Dgkg; S100a8; S100a1; Prf1; Cetn4; Sell; S100a9; S100a6; Prrg4; Ncald"]},{"name":"span","attribs":{"title":"Lgals7; Il10; Cd24a; Epcam; Itga3; Sell"},"children":["Lgals7; Il10; Cd24a; Epcam; Itga3; Sell"]},{"name":"span","attribs":{"title":"Cd9; Il10; Cd109; Serpine2; Cd24a; Cd36; S100a9"},"children":["Cd9; Il10; Cd109; Serpine2; Cd24a; Cd36; S100a9"]},{"name":"span","attribs":{"title":"Oxsr1; Ccl3; Ccl5; Cd9; Ccl4; S100a8; Cd24a; Cxcr5; Itga3; Sell; S100a9"},"children":["Oxsr1; Ccl3; Ccl5; Cd9; Ccl4; S100a8; Cd24a; Cxcr5; Itga3; Sell; S100a9"]},{"name":"span","attribs":{"title":"Dsp; Cd9; Dlg2; Ptch1; Cd24a; Ptpn5; Slc39a14; Epcam; Ddr1; Itga3; Cd36; Ocln; Cd8a; Car4; Arrb1"},"children":["Dsp; Cd9; Dlg2; Ptch1; Cd24a; Ptpn5; Slc39a14; Epcam; Ddr1; Itga3; Cd36; Ocln; Cd8a; Car4; Arrb1"]},{"name":"span","attribs":{"title":"Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Itga3; Cd36; Sell; Cd80; Cd8a; Car4; S100a6"},"children":["Lag3; Cd9; Serpine2; Cd24a; Cxcr5; Itga3; Cd36; Sell; Cd80; Cd8a; Car4; S100a6"]},{"name":"span","attribs":{"title":"Areg; Gcg; Ccl3; Ccl5; Il10; Ccl4; Insl6"},"children":["Areg; Gcg; Ccl3; Ccl5; Il10; Ccl4; Insl6"]},{"name":"span","attribs":{"title":"Twsg1; Cd109; Cd36"},"children":["Twsg1; Cd109; Cd36"]},{"name":"span","attribs":{"title":"Ccl3; Ccl5; Ccl4; S100a8; Sell; S100a9"},"children":["Ccl3; Ccl5; Ccl4; S100a8; Sell; S100a9"]}]}],"filterable":true,"dataKey":"3dd2449c2745aefdcb36a64048e7738d"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="CD49bvDP-up" class="section level3">
<h3 class="tabset">CD49bvDP</h3>
<em>All 1 ontologies considered enriched in the set of 6 up-regulated genes mapped to EntrezIDs from the comparison CD49bvDP. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-8d4006aa37c69a211656" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8d4006aa37c69a211656">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["BP"],"ID":["GO:0050862"],"Description":["positive regulation of T cell receptor signaling pathway"],"Count":[2],"% DE":[0.5],"% BG":[0.0017780566886309],"p.adjust":[0.00816047256941294],"geneID":["Tespa1; Trat1"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["0.008"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Tespa1; Trat1"},"children":["Tespa1; Trat1"]}]}],"filterable":true,"dataKey":"2857201f63046fd9835e32d8a75de611"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="enrichment-results-down-only" class="section level2 tabset">
<h2 class="tabset">Enrichment Results (Down Only)</h2>
<pre class="r"><code>ego_results &lt;- top_tables %&gt;% 
  lapply(
    function(x) {
      de &lt;- dplyr::filter(x, adj.P.Val &lt; 0.05, logFC &lt; -1)
      ids &lt;- unique(unlist(de$entrezid))
      enrichGO(
        ids,
        universe = universe,
        OrgDb = org.Mm.eg.db,
        ont = &quot;ALL&quot;,
        keyType = &quot;ENTREZID&quot;,
        pAdjustMethod = &quot;bonferroni&quot;,
        pvalueCutoff = 0.05,
        qvalueCutoff = 0.05,
        readable = TRUE
      )
    }
  )</code></pre>
<p>Enrichment testing was repeated using all three available Gene
Ontologies and the function <code>enrichGO()</code> from the package
<code>clusterProfiler</code>. Ontologies were only considered to be
enriched amongst the <strong>down-regulated genes</strong> if a
Bonferroni-adjusted p-value &lt; 0.05 was returned during enrichment
testing. Please note that a down-regulated gene does not always
correspond to an decrease in activity of a pathway.</p>
<div id="dn-vs.-lag3-2" class="section level3">
<h3>DN Vs. Lag3<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$DNvLAG3) +
  ggtitle(expression(paste(&quot;DN Vs. Lag&quot;, 3^{textstyle(&quot;+&quot;)}, &quot;(Down Only)&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-dn-lag3-down-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-dn-lag3-down-1">
Past versions of dotplot-dn-lag3-down-1.png
</button>
</p>
<div id="fig-dotplot-dn-lag3-down-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-dn-lag3-down-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dn-vs.-dp-2" class="section level3">
<h3>DN Vs. DP</h3>
<pre class="r"><code>dotplot(ego_results$DNvDP) +
  ggtitle(&quot;DN Vs. DP (Down Only)&quot;) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-dn-dp-down-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-dn-dp-down-1">
Past versions of dotplot-dn-dp-down-1.png
</button>
</p>
<div id="fig-dotplot-dn-dp-down-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-dn-dp-down-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dn-vs.-cd49b-2" class="section level3">
<h3>DN Vs. Cd49b<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$DNvCD49b) +
  ggtitle(expression(paste(&quot;DN Vs. Cd49&quot;, b^{textstyle(&quot;+&quot;)}, &quot;(Down Only)&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-dn-cd49-down-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-dn-cd49-down-1">
Past versions of dotplot-dn-cd49-down-1.png
</button>
</p>
<div id="fig-dotplot-dn-cd49-down-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-dn-cd49-down-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="lag3-vs-cd49-2" class="section level3">
<h3>Lag3<sup>+</sup> Vs Cd49<sup>+</sup></h3>
<pre class="r"><code>dotplot(ego_results$LAG3vCD49b) +
  ggtitle(
    expression(
      paste(&quot;Lag&quot;, 3^{textstyle(&quot;+&quot;)}, &quot; Vs. CD49&quot;, b^{textstyle(&quot;+&quot;)}, &quot;(Down Only)&quot;)
    )
  ) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-lag3-cd49-down-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-lag3-cd49-down-1">
Past versions of dotplot-lag3-cd49-down-1.png
</button>
</p>
<div id="fig-dotplot-lag3-cd49-down-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-lag3-cd49-down-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="lag3-vs.-dp-2" class="section level3">
<h3>Lag3<sup>+</sup> Vs. DP</h3>
<pre class="r"><code>dotplot(ego_results$LAG3vDP) +
  ggtitle(expression(paste(&quot;Lag&quot;, 3^{textstyle(&quot;+&quot;)}, &quot; Vs. DP&quot;, &quot;(Down Only)&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-lag3-dp-down-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-lag3-dp-down-1">
Past versions of dotplot-lag3-dp-down-1.png
</button>
</p>
<div id="fig-dotplot-lag3-dp-down-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-lag3-dp-down-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cd49b-vs.-dp-2" class="section level3">
<h3>Cd49b<sup>+</sup> Vs. DP</h3>
<pre class="r"><code>dotplot(ego_results$CD49bvDP) +
  ggtitle(expression(paste(&quot;Cd49&quot;, b^{textstyle(&quot;+&quot;)}, &quot; Vs. DP (Down Only)&quot;))) +
  scale_y_discrete(labels = label_wrap(25)) +
  theme(
    plot.title = element_text(hjust = 0.5),
    text = element_text(size = 16)
  )</code></pre>
<p><img src="figure/enrichment.Rmd/dotplot-cd49-dp-down-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-cd49-dp-down-1">
Past versions of dotplot-cd49-dp-down-1.png
</button>
</p>
<div id="fig-dotplot-cd49-dp-down-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/smped/Tr1-RNA-Sequencing-/blob/0b3ca1bda6bf71ce73b337b91895ad0962c876c2/docs/figure/enrichment.Rmd/dotplot-cd49-dp-down-1.png" target="_blank">0b3ca1b</a>
</td>
<td>
steveped
</td>
<td>
2022-11-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="enrichment-tables-down-only" class="section level2 tabset">
<h2 class="tabset">Enrichment Tables (Down Only)</h2>
<pre class="r"><code>ego_results &lt;- ego_results[vapply(ego_results, nrow, integer(1)) &gt; 0]
htmltools::tagList(
  names(ego_results) %&gt;% 
    lapply(
      function(x) {
        df &lt;- ego_results[[x]] %&gt;% 
          as_tibble() %&gt;% 
          dplyr::filter(p.adjust &lt; 0.05) %&gt;% 
          mutate(
            geneID = str_replace_all(geneID, &quot;\\/&quot;, &quot;; &quot;),
            `% DE` = vapply(GeneRatio, function(x) eval(parse(text = x)), numeric(1)),
            `% BG` = vapply(BgRatio, function(x) eval(parse(text = x)), numeric(1))
          ) %&gt;% 
          dplyr::select(ONTOLOGY, ID, Description, Count, starts_with(&quot;%&quot;), p.adjust, geneID) %&gt;% 
          arrange(p.adjust) %&gt;% 
          distinct(ONTOLOGY, geneID, .keep_all = TRUE) 
        
        if (nrow(df)) {
          cp &lt;- glue(
            &quot;All {nrow(df)} ontologies considered enriched in the set of &quot;,
            length(unique(unlist(dplyr::filter(top_tables[[x]], DE, logFC &lt; -1)$entrezid))),
            &quot; down-regulated genes mapped to EntrezIDs from the comparison {x}. Enrichment &quot;,
            &quot;testing was performed using Fisher&#39;s Exact Test compared to the set of &quot;,
            comma(length(unique(universe))), 
            &quot; unique EntrezIDs mapped to genes considered as detectable. &quot;,
            &quot;The percentage of nonDE genes mapped to the ontology are also given. &quot;,
            &quot;P-values are Bonferroni adjusted and all pathways are considered enriched &quot;,
            &quot;using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your &quot;, 
            &quot;mouse over the final column. &quot;,
            &quot;Where the identical genes map to multiple terms within each ontology, only &quot;, 
            &quot;the term with the lowest p-value (i.e. strongest enrichment) is shown. &quot;,
            &quot;As a result, some of the pathways seen in the above dotplots may not be &quot;,
            &quot;presented in these tables.&quot;
          )
          htmltools::div(
            htmltools::div(
              id = glue(&quot;{x}-down&quot;), 
              class=&quot;section level3&quot;,
              htmltools::h3(class = &quot;tabset&quot;, x),
              htmltools::tags$em(cp),
              df %&gt;% 
                reactable(
                  filterable = TRUE,
                  columns = list(
                    ONTOLOGY = colDef(name = &quot;Ontology&quot;, maxWidth = 100),
                    ID = colDef(name = &quot;GO ID&quot;, maxWidth = 125),
                    Description = colDef(maxWidth = 250),
                    Count = colDef(name = &quot;N&lt;sub&gt;DE&lt;/sub&gt;&quot;, html = TRUE, maxWidth = 80),
                    `% DE` = colDef(format = colFormat(percent = TRUE, digits = 1), maxWidth = 85),
                    `% BG` = colDef(format = colFormat(percent = TRUE, digits = 1), maxWidth = 85),
                    p.adjust = colDef(
                      name = &quot;p&lt;sub&gt;adj&lt;/sub&gt;&quot;, html = TRUE, maxWidth = 100,
                      cell = function(value) {
                        fm &lt;- ifelse(value &lt; 0.001, &quot;%.2e&quot;, &quot;%.3f&quot;)
                        sprintf(fm, value)
                      }
                    ),
                    geneID = colDef(
                      name = &quot;DE Genes&quot;,
                      cell = function(value) with_tooltip(value, width = 100)
                    )
                  )
                )
            )
          )
        } else {
          cat(glue(&quot;No enrichment was found for {x}\n\n&quot;))
        }
      }
    )
)</code></pre>
<div>
<div id="DNvLAG3-down" class="section level3">
<h3 class="tabset">DNvLAG3</h3>
<em>All 2 ontologies considered enriched in the set of 29 down-regulated genes mapped to EntrezIDs from the comparison DNvLAG3. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-698508e270cc5a2fa6f5" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-698508e270cc5a2fa6f5">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["CC","CC"],"ID":["GO:0098839","GO:0005615"],"Description":["postsynaptic density membrane","extracellular space"],"Count":[3,7],"% DE":[0.111111111111111,0.259259259259259],"% BG":[0.00259094206653539,0.0453933050057001],"p.adjust":[0.00478826442135907,0.0169091943773969],"geneID":["Slc16a3; Dlg2; Cnih2","Fam20a; Areg; Lgals7; Cst7; Ccl5; Serpinf1; Gpld1"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["0.005","0.017"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Slc16a3; Dlg2; Cnih2"},"children":["Slc16a3; Dlg2; Cnih2"]},{"name":"span","attribs":{"title":"Fam20a; Areg; Lgals7; Cst7; Ccl5; Serpinf1; Gpld1"},"children":["Fam20a; Areg; Lgals7; Cst7; Ccl5; Serpinf1; Gpld1"]}]}],"filterable":true,"dataKey":"5869470085154b5a41b5b965a3d2a5de"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="DNvDP-down" class="section level3">
<h3 class="tabset">DNvDP</h3>
<em>All 3 ontologies considered enriched in the set of 33 down-regulated genes mapped to EntrezIDs from the comparison DNvDP. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-4541cb6ae0d17a06834e" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-4541cb6ae0d17a06834e">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["MF","MF","CC"],"ID":["GO:0038023","GO:0004888","GO:0043235"],"Description":["signaling receptor activity","transmembrane signaling receptor activity","receptor complex"],"Count":[8,6,5],"% DE":[0.25,0.1875,0.15625],"% BG":[0.0300877559548684,0.0207898035938153,0.0150274639859053],"p.adjust":[0.000420354573014834,0.00531617305783759,0.0110080871581026],"geneID":["Il12rb2; Tnfrsf9; Klrd1; Gpr15; Cd7; Lag3; Gpr160; Itga2","Il12rb2; Klrd1; Gpr15; Lag3; Gpr160; Itga2","Il12rb2; Klrd1; Cnih2; Gpr160; Itga2"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["4.20e-04","0.005","0.011"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Il12rb2; Tnfrsf9; Klrd1; Gpr15; Cd7; Lag3; Gpr160; Itga2"},"children":["Il12rb2; Tnfrsf9; Klrd1; Gpr15; Cd7; Lag3; Gpr160; Itga2"]},{"name":"span","attribs":{"title":"Il12rb2; Klrd1; Gpr15; Lag3; Gpr160; Itga2"},"children":["Il12rb2; Klrd1; Gpr15; Lag3; Gpr160; Itga2"]},{"name":"span","attribs":{"title":"Il12rb2; Klrd1; Cnih2; Gpr160; Itga2"},"children":["Il12rb2; Klrd1; Cnih2; Gpr160; Itga2"]}]}],"filterable":true,"dataKey":"d71727d784183c199f62b3904d72cff0"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="DNvCD49b-down" class="section level3">
<h3 class="tabset">DNvCD49b</h3>
<em>All 1 ontologies considered enriched in the set of 2 down-regulated genes mapped to EntrezIDs from the comparison DNvCD49b. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-ce73e47b7d96600dab47" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-ce73e47b7d96600dab47">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["MF"],"ID":["GO:0015026"],"Description":["coreceptor activity"],"Count":[1],"% DE":[0.5],"% BG":[0.0014625992478061],"p.adjust":[0.0263089070213729],"geneID":["Gpr15"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["0.026"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Gpr15"},"children":["Gpr15"]}]}],"filterable":true,"dataKey":"1c656bcd83182033c8387f87fe7c3468"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="LAG3vCD49b-down" class="section level3">
<h3 class="tabset">LAG3vCD49b</h3>
<em>All 14 ontologies considered enriched in the set of 53 down-regulated genes mapped to EntrezIDs from the comparison LAG3vCD49b. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-dde7d32fc69217457bc6" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-dde7d32fc69217457bc6">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["MF","MF","CC","MF","CC","CC","CC","BP","BP","CC","MF","BP","BP","BP"],"ID":["GO:0038023","GO:0004888","GO:0009897","GO:0140375","GO:0009986","GO:0098552","GO:0005615","GO:0050856","GO:0006935","GO:0043235","GO:0001968","GO:0002700","GO:0002449","GO:0045785"],"Description":["signaling receptor activity","transmembrane signaling receptor activity","external side of plasma membrane","immune receptor activity","cell surface","side of membrane","extracellular space","regulation of T cell receptor signaling pathway","chemotaxis","receptor complex","fibronectin binding","regulation of production of molecular mediator of immune response","lymphocyte mediated immunity","positive regulation of cell adhesion"],"Count":[13,11,9,6,11,10,11,5,9,6,3,7,8,9],"% DE":[0.26,0.22,0.18,0.12,0.22,0.2,0.22,0.0980392156862745,0.176470588235294,0.12,0.06,0.137254901960784,0.156862745098039,0.176470588235294],"% BG":[0.0300877559548684,0.0207898035938153,0.0218675510415587,0.00679063936481404,0.0403150585552907,0.0353404497875427,0.0453933050057001,0.00439284593661751,0.0257295262001883,0.0150274639859053,0.0014625992478061,0.0145382282188056,0.0231147369522017,0.0320050203953561],"p.adjust":[3.6062391504617e-07,9.54070866761577e-07,0.000117660610067065,0.00021248508693964,0.000380496428821609,0.000822245732789844,0.00118271320772573,0.00431928215895975,0.00843246596669507,0.010145889735127,0.0102206138261903,0.0133601455680206,0.0323913632648885,0.0486442544118965],"geneID":["P2rx7; Lilrb4b; Lilrb4a; Sorl1; Itga2; Ccr4; Gpr15; Vipr1; Ccr3; Klrk1; Reck; Cd40; Cd74","P2rx7; Lilrb4b; Lilrb4a; Sorl1; Itga2; Ccr4; Gpr15; Vipr1; Ccr3; Klrk1; Cd74","P2rx7; Itgb3; Lilrb4a; Itga2; Ccr4; Cd226; Klrk1; Cd40; Cd74","Lilrb4b; Lilrb4a; Ccr4; Ccr3; Klrk1; Cd74","P2rx7; Lilrb4b; Itgb3; Lilrb4a; Sorl1; Itga2; Ccr4; Cd226; Klrk1; Cd40; Cd74","P2rx7; Itgb3; Lilrb4a; Itga2; Ccr4; Cd226; Pkp4; Klrk1; Cd40; Cd74","Sorl1; Cpm; Klk1b9; Ggt1; Bmp7; Tnfsf14; Cxcl16; Ccr3; Klk1b11; Cd40; Cd74","Trat1; Lilrb4b; Tespa1; Lilrb4a; Cd226","Itgb3; Itga2; Ccr4; Egr2; Bmp7; Tnfsf14; Cxcl16; Ccr3; Cd74","Trat1; Itgb3; Itga2; Vipr1; Cd40; Cd74","Lilrb4b; Itgb3; Lilrb4a","P2rx7; Lilrb4b; Lilrb4a; Cd226; Cgas; Cd40; Cd74","P2rx7; Lilrb4b; Lilrb4a; Cd226; Gzmm; Klrk1; Cd40; Cd74","Lilrb4b; Tespa1; Itgb3; Lilrb4a; Itga2; Cass4; Bmp7; Tnfsf14; Cd74"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["3.61e-07","9.54e-07","1.18e-04","2.12e-04","3.80e-04","8.22e-04","0.001","0.004","0.008","0.010","0.010","0.013","0.032","0.049"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Lilrb4a; Sorl1; Itga2; Ccr4; Gpr15; Vipr1; Ccr3; Klrk1; Reck; Cd40; Cd74"},"children":["P2rx7; Lilrb4b; Lilrb4a; Sorl1; Itga2; Ccr4; Gpr15; Vipr1; Ccr3; Klrk1; Reck; Cd40; Cd74"]},{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Lilrb4a; Sorl1; Itga2; Ccr4; Gpr15; Vipr1; Ccr3; Klrk1; Cd74"},"children":["P2rx7; Lilrb4b; Lilrb4a; Sorl1; Itga2; Ccr4; Gpr15; Vipr1; Ccr3; Klrk1; Cd74"]},{"name":"span","attribs":{"title":"P2rx7; Itgb3; Lilrb4a; Itga2; Ccr4; Cd226; Klrk1; Cd40; Cd74"},"children":["P2rx7; Itgb3; Lilrb4a; Itga2; Ccr4; Cd226; Klrk1; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Lilrb4b; Lilrb4a; Ccr4; Ccr3; Klrk1; Cd74"},"children":["Lilrb4b; Lilrb4a; Ccr4; Ccr3; Klrk1; Cd74"]},{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Itgb3; Lilrb4a; Sorl1; Itga2; Ccr4; Cd226; Klrk1; Cd40; Cd74"},"children":["P2rx7; Lilrb4b; Itgb3; Lilrb4a; Sorl1; Itga2; Ccr4; Cd226; Klrk1; Cd40; Cd74"]},{"name":"span","attribs":{"title":"P2rx7; Itgb3; Lilrb4a; Itga2; Ccr4; Cd226; Pkp4; Klrk1; Cd40; Cd74"},"children":["P2rx7; Itgb3; Lilrb4a; Itga2; Ccr4; Cd226; Pkp4; Klrk1; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Sorl1; Cpm; Klk1b9; Ggt1; Bmp7; Tnfsf14; Cxcl16; Ccr3; Klk1b11; Cd40; Cd74"},"children":["Sorl1; Cpm; Klk1b9; Ggt1; Bmp7; Tnfsf14; Cxcl16; Ccr3; Klk1b11; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Trat1; Lilrb4b; Tespa1; Lilrb4a; Cd226"},"children":["Trat1; Lilrb4b; Tespa1; Lilrb4a; Cd226"]},{"name":"span","attribs":{"title":"Itgb3; Itga2; Ccr4; Egr2; Bmp7; Tnfsf14; Cxcl16; Ccr3; Cd74"},"children":["Itgb3; Itga2; Ccr4; Egr2; Bmp7; Tnfsf14; Cxcl16; Ccr3; Cd74"]},{"name":"span","attribs":{"title":"Trat1; Itgb3; Itga2; Vipr1; Cd40; Cd74"},"children":["Trat1; Itgb3; Itga2; Vipr1; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Lilrb4b; Itgb3; Lilrb4a"},"children":["Lilrb4b; Itgb3; Lilrb4a"]},{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Lilrb4a; Cd226; Cgas; Cd40; Cd74"},"children":["P2rx7; Lilrb4b; Lilrb4a; Cd226; Cgas; Cd40; Cd74"]},{"name":"span","attribs":{"title":"P2rx7; Lilrb4b; Lilrb4a; Cd226; Gzmm; Klrk1; Cd40; Cd74"},"children":["P2rx7; Lilrb4b; Lilrb4a; Cd226; Gzmm; Klrk1; Cd40; Cd74"]},{"name":"span","attribs":{"title":"Lilrb4b; Tespa1; Itgb3; Lilrb4a; Itga2; Cass4; Bmp7; Tnfsf14; Cd74"},"children":["Lilrb4b; Tespa1; Itgb3; Lilrb4a; Itga2; Cass4; Bmp7; Tnfsf14; Cd74"]}]}],"filterable":true,"dataKey":"a6c3a3ed1bf2eb71fd05a6abd0be344d"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="CD49bvDP-down" class="section level3">
<h3 class="tabset">CD49bvDP</h3>
<em>All 3 ontologies considered enriched in the set of 28 down-regulated genes mapped to EntrezIDs from the comparison CD49bvDP. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-dd639c94d3274d155b3b" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-dd639c94d3274d155b3b">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["CC","CC","MF"],"ID":["GO:0005615","GO:0001533","GO:0016209"],"Description":["extracellular space","cornified envelope","antioxidant activity"],"Count":[10,3,3],"% DE":[0.37037037037037,0.111111111111111,0.111111111111111],"% BG":[0.0453933050057001,0.0013472898745984,0.00480568324279148],"p.adjust":[1.28424595108818e-05,0.000499186050360298,0.044279687672529],"geneID":["Fam20a; Igfbp7; Lgals7; Areg; S100a8; Cst7; Csf1; S100a9; Pla1a; Prrg4","Lgals7; Dsp; S100a9","S100a8; Prxl2a; S100a9"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["1.28e-05","4.99e-04","0.044"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Fam20a; Igfbp7; Lgals7; Areg; S100a8; Cst7; Csf1; S100a9; Pla1a; Prrg4"},"children":["Fam20a; Igfbp7; Lgals7; Areg; S100a8; Cst7; Csf1; S100a9; Pla1a; Prrg4"]},{"name":"span","attribs":{"title":"Lgals7; Dsp; S100a9"},"children":["Lgals7; Dsp; S100a9"]},{"name":"span","attribs":{"title":"S100a8; Prxl2a; S100a9"},"children":["S100a8; Prxl2a; S100a9"]}]}],"filterable":true,"dataKey":"e216640e46e0ae4c6e8ffd4e44fdf5ce"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div>
<div id="LAG3vDP-down" class="section level3">
<h3 class="tabset">LAG3vDP</h3>
<em>All 2 ontologies considered enriched in the set of 17 down-regulated genes mapped to EntrezIDs from the comparison LAG3vDP. Enrichment testing was performed using Fisher's Exact Test compared to the set of 10,006 unique EntrezIDs mapped to genes considered as detectable. The percentage of nonDE genes mapped to the ontology are also given. P-values are Bonferroni adjusted and all pathways are considered enriched using an adjusted p-value &lt; 0.05. In order to see all DE genes, hover your mouse over the final column. Where the identical genes map to multiple terms within each ontology, only the term with the lowest p-value (i.e. strongest enrichment) is shown. As a result, some of the pathways seen in the above dotplots may not be presented in these tables.</em>
<div class="reactable html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-040c9915e821de4bad75" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-040c9915e821de4bad75">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"ONTOLOGY":["MF","MF"],"ID":["GO:0004888","GO:0004930"],"Description":["transmembrane signaling receptor activity","G protein-coupled receptor activity"],"Count":[6,3],"% DE":[0.352941176470588,0.176470588235294],"% BG":[0.0207898035938153,0.00814876723777685],"p.adjust":[4.90096806714821e-05,0.0208661590063035],"geneID":["Itga2; Klrk1; Gpr15; Vipr1; Gm5127; Cd79b","Gpr15; Vipr1; Gm5127"]},"columns":[{"id":"ONTOLOGY","name":"Ontology","type":"character","maxWidth":100},{"id":"ID","name":"GO ID","type":"character","maxWidth":125},{"id":"Description","name":"Description","type":"character","maxWidth":250},{"id":"Count","name":"N<sub>DE<\/sub>","type":"numeric","html":true,"maxWidth":80},{"id":"% DE","name":"% DE","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"% BG","name":"% BG","type":"numeric","format":{"cell":{"digits":1,"percent":true},"aggregated":{"digits":1,"percent":true}},"maxWidth":85},{"id":"p.adjust","name":"p<sub>adj<\/sub>","type":"numeric","cell":["4.90e-05","0.021"],"html":true,"maxWidth":100},{"id":"geneID","name":"DE Genes","type":"character","cell":[{"name":"span","attribs":{"title":"Itga2; Klrk1; Gpr15; Vipr1; Gm5127; Cd79b"},"children":["Itga2; Klrk1; Gpr15; Vipr1; Gm5127; Cd79b"]},{"name":"span","attribs":{"title":"Gpr15; Vipr1; Gm5127"},"children":["Gpr15; Vipr1; Gm5127"]}]}],"filterable":true,"dataKey":"e94eeb3ddfafbbed19b603e0a37ec847"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.0 (2023-04-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0 
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0

locale:
 [1] LC_CTYPE=en_AU.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_AU.UTF-8        LC_COLLATE=en_AU.UTF-8    
 [5] LC_MONETARY=en_AU.UTF-8    LC_MESSAGES=en_AU.UTF-8   
 [7] LC_PAPER=en_AU.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_AU.UTF-8 LC_IDENTIFICATION=C       

time zone: Australia/Adelaide
tzcode source: system (glibc)

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] scales_1.2.1          glue_1.6.2            htmltools_0.5.5      
 [4] reactable_0.4.4       org.Mm.eg.db_3.17.0   AnnotationDbi_1.62.0 
 [7] IRanges_2.34.0        S4Vectors_0.38.0      Biobase_2.60.0       
[10] BiocGenerics_0.46.0   clusterProfiler_4.8.0 magrittr_2.0.3       
[13] pheatmap_1.0.12       RColorBrewer_1.1-3    edgeR_3.42.0         
[16] limma_3.56.0          lubridate_1.9.2       forcats_1.0.0        
[19] stringr_1.5.0         dplyr_1.1.2           purrr_1.0.1          
[22] readr_2.1.4           tidyr_1.3.0           tibble_3.2.1         
[25] ggplot2_3.4.2         tidyverse_2.0.0       workflowr_1.7.0      

loaded via a namespace (and not attached):
  [1] rstudioapi_0.14         jsonlite_1.8.4          farver_2.1.1           
  [4] rmarkdown_2.21          fs_1.6.2                zlibbioc_1.46.0        
  [7] vctrs_0.6.2             memoise_2.0.1           RCurl_1.98-1.12        
 [10] ggtree_3.8.0            gridGraphics_0.5-1      sass_0.4.5             
 [13] bslib_0.4.2             htmlwidgets_1.6.2       plyr_1.8.8             
 [16] cachem_1.0.7            whisker_0.4.1           igraph_1.4.2           
 [19] lifecycle_1.0.3         pkgconfig_2.0.3         gson_0.1.0             
 [22] Matrix_1.5-4            R6_2.5.1                fastmap_1.1.1          
 [25] GenomeInfoDbData_1.2.10 digest_0.6.31           aplot_0.1.10           
 [28] enrichplot_1.20.0       colorspace_2.1-0        patchwork_1.1.2        
 [31] ps_1.7.5                rprojroot_2.0.3         crosstalk_1.2.0        
 [34] RSQLite_2.3.1           reactR_0.4.4            labeling_0.4.2         
 [37] fansi_1.0.4             timechange_0.2.0        httr_1.4.5             
 [40] polyclip_1.10-4         compiler_4.3.0          here_1.0.1             
 [43] bit64_4.0.5             withr_2.5.0             downloader_0.4         
 [46] BiocParallel_1.34.0     viridis_0.6.2           DBI_1.1.3              
 [49] highr_0.10              ggforce_0.4.1           MASS_7.3-59            
 [52] HDO.db_0.99.1           tools_4.3.0             scatterpie_0.1.9       
 [55] ape_5.7-1               httpuv_1.6.9            callr_3.7.3            
 [58] nlme_3.1-162            GOSemSim_2.26.0         promises_1.2.0.1       
 [61] shadowtext_0.1.2        grid_4.3.0              getPass_0.2-2          
 [64] reshape2_1.4.4          fgsea_1.26.0            generics_0.1.3         
 [67] gtable_0.3.3            tzdb_0.3.0              data.table_1.14.8      
 [70] hms_1.1.3               tidygraph_1.2.3         utf8_1.2.3             
 [73] XVector_0.40.0          ggrepel_0.9.3           pillar_1.9.0           
 [76] yulab.utils_0.0.6       later_1.3.0             splines_4.3.0          
 [79] tweenr_2.0.2            treeio_1.24.0           lattice_0.21-8         
 [82] bit_4.0.5               tidyselect_1.2.0        GO.db_3.17.0           
 [85] locfit_1.5-9.7          Biostrings_2.68.0       knitr_1.42             
 [88] git2r_0.32.0            gridExtra_2.3           xfun_0.39              
 [91] graphlayouts_0.8.4      stringi_1.7.12          lazyeval_0.2.2         
 [94] ggfun_0.0.9             yaml_2.3.7              evaluate_0.20          
 [97] codetools_0.2-19        ggraph_2.1.0            qvalue_2.32.0          
[100] ggplotify_0.1.0         cli_3.6.1               munsell_0.5.0          
[103] processx_3.8.1          jquerylib_0.1.4         Rcpp_1.0.10            
[106] GenomeInfoDb_1.36.0     png_0.1-8               parallel_4.3.0         
[109] ellipsis_0.3.2          blob_1.2.4              DOSE_3.26.0            
[112] bitops_1.0-7            tidytree_0.4.2          viridisLite_0.4.1      
[115] crayon_1.5.2            rlang_1.1.0             cowplot_1.1.1          
[118] fastmatch_1.1-3         KEGGREST_1.40.0        </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
