---
title: "Tr1: Differential Gene Expression"
author: "Stephen Pederson & Caitlin Abbott"
date: "`r format(Sys.Date(), '%d %B, %Y')`"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

```{r setup}
knitr::opts_chunk$set(
  warning = FALSE, message = FALSE,
  fig.width = 10, fig.height = 8,
  dev = c("png", "pdf")
)
```


```{r packages}
library(tidyverse)
library(magrittr)
library(limma)
library(edgeR)
library(ggrepel)
library(ggplot2)
library(pheatmap)
library(AnnotationHub)
library(ensembldb)
library(scales)
library(broom)
library(glue)
library(pander)
library(grid)
library(ComplexUpset)
library(reactable)
theme_set(
  theme_bw() +
    theme(plot.title = element_text(hjust = 0.5))
)
```

## Data Preparation & Inspection {.tabset}

```{r setup-data}
counts <- here::here("data", "genes.out.gz") %>% 
  read_tsv() %>% 
  rename_all(basename) %>% 
  rename_all(str_remove_all, pattern = "Aligned.sortedByCoord.out.bam")
samples <- tibble(
  sample = setdiff(colnames(counts), "Geneid"),
  condition = gsub("M2_|M3_|M5_|M6_", "", sample) %>% 
    factor(levels = c("DN", "DP", "LAG_3", "CD49b")),
  mouse = str_extract(sample, "M[0-9]"),
  label = glue("{condition} ({mouse})")
)
dgeList <- counts %>%
  as.data.frame() %>%
  column_to_rownames("Geneid") %>%
  as.matrix() %>%
  DGEList(
    samples = samples
  ) %>% 
  calcNormFactors(method = "TMM")
```

```{r setup-annotations}
ah <- AnnotationHub()
#Find which genome did we use
# unique(ah$dataprovider)
# subset(ah, dataprovider == "Ensembl") %>%  #In Ensembl databse
#   subset(species == "Mus musculus") %>%  #under Mouse
#   subset(rdataclass == "EnsDb") 
ensDb <- ah[["AH69210"]] #This is the genome used for assigning reads to genes
genes <- genes(ensDb) %>% #extract the genes
  subset(seqnames %in% c(1:19, "MT", "X", "Y")) %>%
  keepStandardChromosomes() %>% 
  sortSeqlevels()
dgeList$genes <- data.frame(gene_id = rownames(dgeList)) %>% 
  left_join(
    mcols(genes) %>% 
      as.data.frame() %>% 
      dplyr::select(gene_id, gene_name, gene_biotype, description, entrezid),
    by = "gene_id"
  )
id2gene <- setNames(genes$gene_name, genes$gene_id)
```


```{r filter-dge}
genes2keep <- dgeList %>% 
  cpm(log = TRUE) %>% 
  is_greater_than(1) %>% 
  rowSums() %>% 
  is_greater_than(4)
dgeFilt <- dgeList[genes2keep,, keep.lib.sizes = FALSE]
```

Genes were only retained in the final dataset if $> 4$ samples returned $>1$ log~2~ Counts per Million (logCPM).
The gave a dataset of `r comma(sum(genes2keep))` of the initial `r comma(length(genes2keep))` genes which were retained for downstream analysis.

### Library Sizes

```{r plot-lib-sizes, fig.height=6, fig.cap = "**Figure S7a** *Library sizes for all libraries after summarisation to gene-level counts. The mean library size across all libraries is shown as a dashed horizontal line.*"}
dgeList$samples %>%
  mutate(CellType = dgeList$samples$condition) %>%
  ggplot(aes(x = label, y = lib.size / 1e6, fill = CellType)) +
  geom_col() +
  geom_hline(
    yintercept = mean(dgeList$samples$lib.size / 1e6),
    linetype = 2, colour = "grey30"
  ) +
  labs(x = "Sample", y = "Library Size (millions)") +
  facet_wrap(~mouse, scales = "free_x", nrow = 1) +
  scale_x_discrete(labels = label_wrap(5)) +
  scale_y_continuous(expand = expansion(c(0, 0.05))) 
```

### Count Densities

```{r plot-densities, fig.cap = "*logCPM densisties after removal of undetectable genes. The double negative (DN) samples for both m% and M6 appear to skew to lower overall counts compared to all other samples, with a handful of highly expressed genes likely to dominate the sample.*"}
dgeFilt %>% 
  cpm(log = TRUE) %>% 
  as_tibble(rownames = "gene_id") %>% 
  pivot_longer(cols = all_of(colnames(dgeFilt)), names_to = "sample", values_to = "logCPM") %>% 
  left_join(dgeFilt$samples, by = "sample") %>% 
  ggplot(aes(logCPM, y = stat(density), colour = condition, group = sample)) +
  geom_density() +
  facet_wrap(~mouse) +
  scale_y_continuous(expand = expansion(c(0.01, 0.05)))
```

### PCA

```{r plot-pca, fig.cap = "**Figure S7b** *PCA on logCPM values, with the two DN samples identified above clearly showing strong divergence from the remainder of the dataset. The CD49b sample fro M2 also appeared slightly divergent, with the previous density plot also showing a slght skew towards lower overall counts.*"}
pca <- dgeFilt %>% 
  cpm(log = TRUE) %>% 
  t() %>% 
  prcomp()
pca %>% 
  broom::tidy() %>% 
  dplyr::rename(sample = row) %>% 
  dplyr::filter(PC %in% c(1, 2)) %>% 
  pivot_wider(names_from = "PC", names_prefix = "PC", values_from = "value") %>% 
  left_join(dgeFilt$samples, by = "sample") %>% 
  ggplot(aes(PC1, PC2, colour = condition)) +
  geom_point(size = 3) +
  geom_text_repel(
    aes(label = label),#str_replace_all(label, " ", "\n")),
    max.overlaps = Inf
  ) +
  labs(
    x = glue("PC1 ({percent(summary(pca)$importance[2, 'PC1'])})"),
    y = glue("PC2 ({percent(summary(pca)$importance[2, 'PC2'])})"),
    colour = "Cell Type"
  )
```

The above PCA revealed some potential problems with two of the four DN samples.
Exclusion of the clear outliers will reduce the number of viable samples within the DN group to 2 and as an alternative, a weighting strategy was instead sought for all samples.

## Model Fitting

```{r get-weights}
U <- matrix(1, nrow = ncol(dgeFilt)) %>% 
  set_colnames("Intercept")
v <- voomWithQualityWeights(dgeFilt, design = U)
X <- model.matrix(~0 + condition, data = dgeFilt$samples) %>% 
  set_colnames(str_remove(colnames(.), "condition"))
rho <- duplicateCorrelation(v, X, block=dgeFilt$samples$mouse)$consensus.correlation
v <- voomWithQualityWeights(
  counts = dgeFilt, design = U, block=dgeFilt$samples$mouse, correlation=rho
)
v$design <- X
```

Sample-level weights were estimated by assuming all samples were drawn from the same group and running `voomWithQualityWeights()`.
After running this, samples were compared within each mouse-of-origin and correlations within mice were estimated using `duplicateCorrelation()` ($\rho=$ `r round(rho, 3)`).
`voomWithQualityWeights()` was then run again setting mouse as the blocking variable and including the consensus correlations

```{r pca2, eval = FALSE, echo=FALSE}
pca2 <- v$E %>% 
  t() %>% 
  prcomp()
pca2 %>% 
  broom::tidy() %>% 
  dplyr::rename(sample = row) %>% 
  dplyr::filter(PC %in% c(1, 2)) %>% 
  pivot_wider(names_from = "PC", names_prefix = "PC", values_from = "value") %>% 
  left_join(v$targets, by = "sample") %>% 
  ggplot(aes(PC1, PC2, colour = condition)) +
  geom_point(aes(size = sample.weights)) +
  geom_text_repel(aes(label = sample)) +
  labs(
    x = glue("PC1 ({percent(summary(pca2)$importance[2, 'PC1'])})"),
    y = glue("PC2 ({percent(summary(pca2)$importance[2, 'PC2'])})"),
    colour = "Cell Type"
  )
```


```{r plot-weights, fig.height=6, fig.cap = "*Sample-level weights after running `voomWithQualityWeights` setting all samples as being drawn from the same condition. The ideal equal wweighting of 1 is shown as the dashed horizontal line, with those samples below this being assumed to be of lower quality than those above the line. Thw two previously identified DN samples were strongly down-weighted, as was the CD49b sample from M2*"}
v$targets %>% 
  ggplot(aes(label, sample.weights, fill = condition)) +
  geom_col() +
  geom_hline(yintercept = 1, linetype = 2, col = "grey30") +
  facet_wrap(~mouse, nrow = 1, scales = "free_x") +
  scale_x_discrete(labels = scales::label_wrap(5)) +
  scale_y_continuous(expand = expansion(c(0, 0.05))) +
  labs(
    x = "Sample", y = "Sample Weights", fill = "Cell Type"
  )
```


```{r fit}
cont.matrix <- makeContrasts(
  DNvLAG3 = DN - LAG_3,
  DNvDP = DN - DP,
  DNvCD49b = DN - CD49b,
  LAG3vCD49b = LAG_3 - CD49b,
  CD49bvDP = CD49b - DP,
  LAG3vDP = LAG_3 - DP,
  levels = X
)
fit <- lmFit(v, design = X, block = v$targets$mouse, correlation = rho) %>% 
  contrasts.fit(cont.matrix) %>%
  # treat()
  eBayes()
```

### Summary Table

```{r top-tables}
top_tables <- colnames(cont.matrix) %>% 
  lapply(function(x) topTable(fit, coef = x, number = Inf)) %>%
  # lapply(function(x) topTreat(fit, coef = x, number = Inf)) %>%
  lapply(as_tibble) %>%
  lapply(mutate, DE = adj.P.Val < 0.05 & abs(logFC) > 1) %>% 
  setNames(colnames(cont.matrix))
top_tables %>% 
  lapply(
    function(x){
      df <- dplyr::filter(x,DE)
      tibble(
        Up = sum(df$logFC > 0),
        Down = sum(df$logFC < 0),
        `Total DE` = Up + Down
      )
    }
  ) %>% 
  bind_rows(.id = "Comparison") %>% 
  pander(
    justify = "lrrr",
    caption = glue(
      "
      Results from each comparison, where genes are considered DE using an FDR
      < 0.05 along with an estimated logFC beyond $\\pm1$. In total, 
      {length(unique(dplyr::filter(bind_rows(top_tables), DE)$gene_id))} 
      unique genes were considered to be DE in at least one comparison.
      "
    )
  )
```

### MA Plots {.tabset}

#### DN Vs. LAG3

```{r plot-ma-dnvlag3, fig.cap = "MA-Plot for DN Vs. LAG3. The 5 most up/down-regulated genes are labelled, with the blue line representing a spline through the data."}
top_tables$DNvLAG3 %>% 
  ggplot(aes(AveExpr, logFC)) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_smooth(se = FALSE) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC > 0) %>% 
      arrange(desc(logFC)) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC < 0) %>% 
      arrange(logFC) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("DN Vs. LAG3") +
  scale_colour_manual(values = c("grey30", "red"))
```

#### DN Vs. DP

```{r plot-ma-dnvdp, fig.cap = "MA-Plot for DN Vs. DP. The 5 most up/down-regulated genes are labelled, with the blue line representing a spline through the data."}
top_tables$DNvDP %>% 
  ggplot(aes(AveExpr, logFC)) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_smooth(se = FALSE) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC > 0) %>% 
      arrange(desc(logFC)) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC < 0) %>% 
      arrange(logFC) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("DN Vs. DP") +
  scale_colour_manual(values = c("grey30", "red"))
```

#### DN Vs. CD49b

```{r plot-ma-dnvcd49, fig.cap = "MA-Plot for DN Vs. CD49b The most up/down-regulated genes are labelled, with the blue line representing a spline through the data."}
top_tables$DNvCD49b %>% 
  ggplot(aes(AveExpr, logFC)) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_smooth(se = FALSE) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC > 0) %>% 
      arrange(desc(logFC)) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC < 0) %>% 
      arrange(logFC) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("DN Vs. CD49b") +
  scale_colour_manual(values = c("grey30", "red"))
```

#### LAG3 Vs. CD49b

```{r plot-ma-lag3vcd49, fig.cap = "MA-Plot for LAG3 Vs. CD49b. The 5 most up/down-regulated genes are labelled, with the blue line representing a spline through the data."}
top_tables$LAG3vCD49b %>% 
  ggplot(aes(AveExpr, logFC)) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_smooth(se = FALSE) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC > 0) %>% 
      arrange(desc(logFC)) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC < 0) %>% 
      arrange(logFC) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("LAG3 Vs. CD49b") +
  scale_colour_manual(values = c("grey30", "red"))
```

#### LAG3 Vs. DP

```{r plot-ma-lag3vdp, fig.cap = "MA-Plot for LAG3 Vs. DP The 5 most up/down-regulated genes are labelled, with the blue line representing a spline through the data."}
top_tables$LAG3vDP %>% 
  ggplot(aes(AveExpr, logFC)) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_smooth(se = FALSE) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC > 0) %>% 
      arrange(desc(logFC)) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC < 0) %>% 
      arrange(logFC) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("LAG3 Vs. DP") +
  scale_colour_manual(values = c("grey30", "red"))
```

#### CD49b Vs. DP

```{r plot-ma-cd49vdp, fig.cap = "MA-Plot for CD49b Vs. DP. The 5 most up/down-regulated genes are labelled, with the blue line representing a spline through the data."}
top_tables$CD49bvDP %>% 
  ggplot(aes(AveExpr, logFC)) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_smooth(se = FALSE) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC > 0) %>% 
      arrange(desc(logFC)) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE, logFC < 0) %>% 
      arrange(logFC) %>% 
      dplyr::slice(1:5),
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("CD49b Vs. DP") +
  scale_colour_manual(values = c("grey30", "red"))
```

### Volcano Plots {.tabset}

#### DN Vs. LAG3

```{r plot-volcano-dnvlag3, fig.cap = "Volcano Plot for DN Vs. LAG3. The (up to) 20 most highly-ranked genes are labelled."}
top_tables$DNvLAG3 %>% 
  ggplot(aes(logFC, -log10(P.Value))) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE) %>% 
      arrange(P.Value) %>% 
      dplyr::slice(1:20),
    size = 5,
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("DN Vs. LAG3") +
  scale_colour_manual(values = c("grey30", "red")) +
  theme(text = element_text(size = 16))
```

#### DN Vs. DP

```{r plot-volcano-dnvdp, fig.cap = "Volcano Plot for DN Vs. DP. The (up to) 20 most highly-ranked genes are labelled."}
top_tables$DNvDP %>% 
  ggplot(aes(logFC, -log10(P.Value))) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE) %>% 
      arrange(P.Value) %>% 
      dplyr::slice(1:20),
    size = 5,
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("DN Vs. DP") +
  scale_colour_manual(values = c("grey30", "red")) +
  theme(text = element_text(size = 16))
```

#### DN Vs. CD49b

```{r plot-volcano-dnvcd49, fig.cap = "Volcano Plot for DN Vs. CD49b. The (up to) 20 most highly-ranked genes are labelled."}
top_tables$DNvCD49b %>% 
  ggplot(aes(logFC, -log10(P.Value))) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE) %>% 
      arrange(P.Value) %>% 
      dplyr::slice(1:20),
    size = 5,
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("DN Vs. CD49b") +
  scale_colour_manual(values = c("grey30", "red")) +
  theme(text = element_text(size = 16))
```

#### LAG3 Vs. CD49b

```{r plot-volcano-lag3vcd49, fig.cap = "Volcano Plot for LAG3 Vs. CD49b. The (up to) 20 most highly-ranked genes are labelled."}
top_tables$LAG3vCD49b %>% 
  ggplot(aes(logFC, -log10(P.Value))) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE) %>% 
      arrange(P.Value) %>% 
      dplyr::slice(1:20),
    size = 5,
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("LAG3 Vs. CD49b") +
  scale_colour_manual(values = c("grey30", "red")) +
  theme(text = element_text(size = 16))
```

#### LAG3 Vs. DP

```{r plot-volcano-lag3vdp, fig.cap = "Volcano Plot for LAG3 Vs. DP. The (up to) 20 most highly-ranked genes are labelled."}
top_tables$LAG3vDP %>% 
  ggplot(aes(logFC, -log10(P.Value))) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE) %>% 
      arrange(P.Value) %>% 
      dplyr::slice(1:20),
    size = 5,
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("LAG3 Vs. DP") +
  scale_colour_manual(values = c("grey30", "red")) +
  theme(text = element_text(size = 16))
```

#### CD49b Vs. DP

```{r plot-volcano-cd49vdp, fig.cap = "Volcano Plot for CD49b Vs. DP. The (up to) 20 most highly-ranked genes are labelled."}
top_tables$CD49bvDP %>% 
  ggplot(aes(logFC, -log10(P.Value))) +
  geom_point(aes(colour = DE), alpha = 0.6) +
  geom_label_repel(
    aes(label = gene_name, colour = DE),
    data = . %>% 
      dplyr::filter(DE) %>% 
      arrange(P.Value) %>% 
      dplyr::slice(1:20),
    size = 5,
    max.overlaps = Inf,
    fontface = "italic",
    show.legend = FALSE
  ) +
  ggtitle("CD49b Vs. DP") +
  scale_colour_manual(values = c("grey30", "red")) +
  theme(text = element_text(size = 16))
```

### DE Gene Summary {.tabset}

#### UpSet Plot

```{r upset-all-de, fig.height=7, fig.cap = "UpSet plot for all DE genes. A complete list of DE genes was obtained by finding all genes considered DE after filtering by FDR and logFC across all comparisons. For the purposes of comparison, any genes in this complete list were considered as DE in a comparison if receiving an FDR-adjusted p-value < 0.05 in order for this figure to give a more accurate picture across all 6 comparisons, and avoiding any misleading results from the use of a hard cutoff. The genes DE in all LAG3 comparisons are highlighted in red, whilst the CD49b signature is shown in green and the DN signature is shown in blue. No clear DP signature was evident in this viewpoint."}
all_de <- top_tables %>% 
  lapply(dplyr::filter, DE) %>%
  bind_rows() %>% 
  pull("gene_id") %>% 
  unique()
top_tables %>% 
  lapply(dplyr::filter, adj.P.Val < 0.05, gene_id %in% all_de) %>%
  lapply(dplyr::select, gene_id) %>% 
  bind_rows(.id = "comparison") %>% 
  mutate(DE = TRUE, comparison = str_replace_all(comparison, "v", " Vs. ")) %>% 
  pivot_wider(names_from = comparison, values_from = DE, values_fill = FALSE) %>% 
  upset(
    intersect = str_replace_all(names(top_tables), "v", " Vs. "),
    base_annotations = list(
      'Intersection size' = intersection_size(bar_number_threshold = 1, width = 0.9) +
        scale_y_continuous(expand = expansion(c(0, 0.1))) +
        scale_fill_manual(values = c(bars_color = "grey20")) +
        theme(panel.grid = element_blank())
    ),
    sort_sets = FALSE,
    set_sizes = upset_set_size() +
      geom_text(aes(label = comma(after_stat(count))), stat = 'count', hjust = 1.1) +
      scale_y_reverse(expand = expansion(c(0.25, 0))) +
      theme(panel.grid = element_blank()),
    queries = list(
      upset_query(
        intersect = str_subset(colnames(.), "LAG3"),
        fill = "red", color = "red",
        only_components = c("intersections_matrix", "Intersection size")
      ),
      upset_query(
        intersect = str_subset(colnames(.), "CD49"),
        fill = "forestgreen", color = "forestgreen",
        only_components = c("intersections_matrix", "Intersection size")
      ),
      upset_query(
        intersect = str_subset(colnames(.), "DN"),
        fill = "blue", color = "blue",
        only_components = c("intersections_matrix", "Intersection size")
      )
    ),
    min_size = 2
  ) +
  labs(x = "Group") +
  theme(panel.grid = element_blank())
```



## Heatmaps {.tabset}

In order to visualise the data using heatmaps, the average expression within each cell type was calculated.

```{r grp-coef}
grp_coef <- dgeList %>%
  cpm(log = TRUE) %>% 
  as_tibble(rownames = "gene_id") %>% 
  pivot_longer(
    cols = all_of(colnames(v)), names_to = "sample", values_to = "logCPM"
  ) %>% 
  left_join(v$targets) %>% 
  group_by(gene_id, condition) %>% 
  # summarise(logCPM = weighted.mean(logCPM, sample.weights)) %>%
  summarise(logCPM = mean(logCPM)) %>%
  pivot_wider(names_from = "condition", values_from = "logCPM") %>% 
  as.data.frame() %>%
  column_to_rownames("gene_id") %>% 
  as.matrix()
```


### Key Signatures

```{r sig-heat, results='hide'}
signatures <- top_tables %>% 
  bind_rows(.id = "comparison") %>% 
  dplyr::filter(gene_id %in% all_de, adj.P.Val < 0.05) %>% 
  dplyr::select(comparison, gene_id, gene_name, AveExpr) %>% 
  chop(comparison) %>% 
  dplyr::filter(vapply(comparison, length, integer(1)) == 3) %>% 
  dplyr::mutate(
    Signature = case_when(
      vapply(comparison, function(x) sum(str_detect(x, "CD49")) == 3, logical(1)) ~ "Cd49b+",
      vapply(comparison, function(x) sum(str_detect(x, "LAG3")) == 3, logical(1)) ~ "Lag3+",
      vapply(comparison, function(x) sum(str_detect(x, "DN")) == 3, logical(1)) ~ "DN",
      TRUE ~ "Other"
    )
  ) %>% 
  dplyr::filter(Signature != "Other") %>% 
  dplyr::select(starts_with("gene"), Signature, AveExpr) %>% 
  arrange(Signature, desc(AveExpr)) %>% 
  as.data.frame() %>% 
  column_to_rownames("gene_id")
sig_heat <- grp_coef[rownames(signatures),] %>% 
  pheatmap(
    annotation_row = dplyr::select(signatures, Signature),
    cluster_rows = FALSE, cluster_cols = FALSE,
    labels_row = setNames(signatures$gene_name, rownames(signatures)),
    labels_col = c(DN = "DN", DP = "DP", LAG_3 = "Lag3+", CD49b = "CD49b+"),
    gaps_row = cumsum(c(
      sum(signatures$Signature == "Cd49b+"),
      sum(signatures$Signature == "DN")
    )),
    color = hcl.colors(101, "inferno"),
    cutree_rows = 7,
    cellwidth = 25,
    cellheight = 15,
    annotation_colors = list(
      Signature = c("Cd49b+" = "forestgreen", "DN" = "blue", "Lag3+" = "red")
    ),
    silent = TRUE
  ) %>% 
  .[["gtable"]]
sig_heat$grobs[[3]]$gp <- gpar(fonsize = 10, fontface = "italic")
png(
  here::here("docs", "assets", "sig_heat.png"),
  height = 6, width = 4, units = "in", res = 300
)
grid.newpage()
grid.draw(sig_heat)
dev.off()
pdf(
  here::here("docs", "assets", "sig_heat.pdf"),
  height = 6, width = 4
)
grid.newpage()
grid.draw(sig_heat)
dev.off()
```

![Expression values from genes in each of the key signatures defined in the previous UpSet plot were plotted. Within each signature genes are arranged in order of average expression. Most genes in the CD49b+ signature showed lower expression in this cell type, whilst patterns were more varied for each of the other signatures. The pdf of this image is available [here](assets/sig_heat.pdf)](assets/sig_heat.png){width=400px}

### High Confidence Genes

```{r make-highconf-heat, results='hide'}
highConf_df <- tribble(
  ~Group, ~gene_name,
  "Treg", "Foxp3",
  "Treg", "Il2ra",
  "Treg", "Il2rb",
  "Treg", "Ikzf4",
  "Treg", "Il1rl1",
  "Th2", "Gata3",
  "Th2", "Il4",
  "Th2", "Il5",
  "Th2", "Il13",
  "Th17", "Rorc",
  "Th17", "Il17a",
  "Th17", "Il17f",
  "Th17", "Ccr6",
  "Th1", "Tbx21",
  "Th1", "Cxcr3",
  "Th1", "Cxcr6",
  "Th1", "Tnf",
  "Th1", "Ifng",
  "Tfh", "Bcl6",
  "Tfh", "Cxcr5",
  "Tfh", "Tcf7",
  "Tfh", "Tox2",
  "Tr1", "Ccr5",
  "Tr1", "Il10",
  "Tr1", "Il21",
  "Tr1", "Tgfb1",
  "Tr1", "Gzmb",
  "Tr1", "Lag3",
  "Tr1", "Itga2",
  "Tr1", "Icos",
  "Tr1", "Pdcd1",
  "Tr1", "Ctla4",
  "Tr1", "Havcr2",
  "Tr1", "Tigit",
  "Tr1", "Cd226",
  "Tr1", "Maf",
  "Tr1", "Prdm1",
  "Tr1", "Batf",
  "Tr1", "Ahr",
  "Tr1", "Eomes",
  "Tr1", "Irf1",
  "Tr1", "Irf4",
  "Tr1", "Stat1",
  "Tr1", "Stat3"
) %>%
  left_join(
    # dplyr::select(dgeFilt$genes, gene_name, gene_id)
    dplyr::select(dgeList$genes, gene_name, gene_id)
  ) %>% 
  as.data.frame() %>% 
  column_to_rownames("gene_id")
p <- grp_coef[rownames(highConf_df),] %>% 
  pheatmap(
    cluster_rows = FALSE, cluster_cols = FALSE,
    labels_row = setNames(highConf_df$gene_name, rownames(highConf_df)),
    labels_col = c(DN = "DN", DP = "DP", LAG_3 = "Lag3+", CD49b = "CD49b+"),
    color = hcl.colors(101, "inferno"),
    cellwidth = 15,
    silent = TRUE
    # annotation_col = dplyr::select(highConf_df, Group)
  ) %>% 
  .[["gtable"]]
p$grobs[[3]]$gp <- gpar(fontsize = 10, fontface = "italic")
bu <- 0.07
grp_df <- highConf_df %>%
  mutate(i = seq_along(gene_name) / nrow(.)) %>% 
  group_by(Group) %>% 
  summarise(
    y_max = 1 - min(i) + 0.004,
    y_min = 1 - max(i) - 0.003
  ) %>% 
  mutate(
    y_max = bu + (1 - bu) * y_max,
    y_min = bu + (1 - bu) * y_min
  ) %>% 
  split(.$Group)
png(
  here::here("docs", "assets", "highconf_heat.png"),
  width = 5, height = 10, res = 300, units = "in"
)
grid.newpage()
grid.draw(p)
grp_df %>% 
  lapply(
    function(x) {
      grid.lines(
        x = 0.305, 
        y = c(x$y_max, x$y_min), 
        gp = gpar(lwd = 2.5)
      )
      grid.text(
        x$Group, x = 0.25,
        y = 0.5 * (x$y_min + x$y_max) + 0.01
      )
    }
  )
dev.off()
pdf(
  here::here("docs", "assets", "highconf_heat.pdf"),
  width = 5, height = 10
)
grid.newpage()
grid.draw(p)
grp_df %>% 
  lapply(
    function(x) {
      grid.lines(
        x = 0.305, 
        y = c(x$y_max, x$y_min), 
        gp = gpar(lwd = 2.5)
      )
      grid.text(
        x$Group, x = 0.25,
        y = 0.5 * (x$y_min + x$y_max) + 0.01
      )
    }
  )
dev.off()
```

![Expression patterns across all groups for key marker genes. The same figure is available [here](assets/highconf_heat.pdf) as a pdf.](assets/highconf_heat.png){width=300px}

### Top 100

All DE genes were combined across all comparisons and the 100 with the most extreme estimates of logFC were chosen for visualisation.

```{r heatmap-top100, results='hide', fig.height=4, fig.cap="The 100 genes showing the most extreme fold-change across one or more comparisons"}
top_100 <- top_tables %>% 
  lapply(dplyr::filter, DE) %>% 
  bind_rows(.id = "comparison") %>% 
  arrange(desc(abs(logFC))) %>% 
  distinct(gene_id) %>% 
  dplyr::slice(1:100) %>% 
  pull("gene_id")
p <- grp_coef[top_100,] %>% 
  # t() %>% 
  pheatmap(
    color = hcl.colors(101, "inferno"),
    labels_row = id2gene[top_100],
    labels_col = c(DN = "DN", DP = "DP", "LAG_3" = "Lag3+", CD49b = "Cd49b+"),
    cluster_cols = FALSE,
    # annotation_col = top_tables %>%
    #   lapply(
    #     function(x) {
    #       up <- dplyr::filter(x, DE, logFC > 0)$gene_id
    #       down <- dplyr::filter(x, DE, logFC < 0)$gene_id
    #       case_when(
    #         top_100 %in% up ~ "Up",
    #         top_100 %in% down ~ "Down",
    #         TRUE ~ "Unchanged"
    #       )
    #     }
    #   ) %>%
    #   lapply(factor, levels = c("Up", "Down", "Unchanged")) %>%
    #   as.data.frame() %>%
    #   set_colnames(str_replace_all(colnames(.), "v", " Vs. ")) %>%
    #   set_rownames(top_100),
    # annotation_colors = top_tables %>%
    #   setNames(str_replace_all(names(.), "v", " Vs. ")) %>%
    #   lapply(function(x) c("Unchanged" = "grey", "Up" = "red", "Down" = "blue")),
    # annotation_legend = FALSE,
    cutree_rows = 8,
    cellwidth = 15,
    fontsize = 8,
    silent = TRUE
  ) %>% 
  .[["gtable"]]
p$grobs[[4]]$gp <- gpar(fontsize = 8, fontface = "italic")
p$grobs[[3]]$gp <- gpar(fontsize = 10)
png(
  here::here("docs", "assets", "top100_heat.png"),
  height = 12, width = 5, units = "in", res = 300
)
grid.newpage()
grid.draw(p)
dev.off()
pdf(
  here::here("docs", "assets", "top100_heat.pdf"),
  height = 12, width = 5
)
grid.newpage()
grid.draw(p)
dev.off()
```

![Expression patterns the top 100 DE genes by logFC. The same figure is available [here](assets/top100_heat.pdf) as a pdf.](assets/top100_heat.png){width=300px}

### Top 100 By logFC

```{r heatmap-top100-logfc, results='hide'}
n <- 101
myPalette <- colorRampPalette(c("blue", "white", "red"))(n)
myBreaks <- c(
  seq(-7, -7/n, length.out = (n - 1) / 2), 
  0,
  seq(9 / n, 9, length.out = (n - 1) / 2)
)
p <- fit$coefficients[top_100,c(1:4, 6, 5)] %>% 
  pheatmap(
    color = myPalette,
    breaks = myBreaks,
    legend_breaks = seq(-6, 8, by = 2),
    cellwidth = 15,
    labels_row = id2gene[top_100],
    cutree_rows = 5,
    fontsize = 8,
    cluster_cols = FALSE,
    labels_col = setNames(
      str_replace(colnames(.), "v", " Vs. "),
      colnames(.)
    ),
    silent = TRUE
  ) %>% 
  .[["gtable"]]
p$grobs[[4]]$gp <- gpar(fontface = "italic")
png(
  here::here("docs", "assets", "top100_logfc_heat.png"),
  height = 12, width = 5, units = "in", res = 300
)
grid.newpage()
grid.draw(p)
dev.off()
pdf(
  here::here("docs", "assets", "top100_logfc_heat.pdf"),
  height = 12, width = 5
)
grid.newpage()
grid.draw(p)
dev.off()
```


![Changes in relative expression for the top 100 DE genes by logFC. The same figure is available [here](assets/top100_logfc_heat.pdf) as a pdf.](assets/top100_logfc_heat.png){width=300px}

## Data Export

```{r export}
write_rds(dgeFilt, here::here("output", "dgeFilt.rds"), compress = "gz")
write_rds(top_tables, here::here("output", "top_tables.rds"), compress = "gz")
write_rds(v, here::here("output", "v.rds"), compress = "gz")
write_rds(fit, here::here("output", "fit.rds"), compress = "gz")
```
